\chapter{Evaluation of the Delivery of RAR Inverse Agonists with a Coiled-coil
Protein}

\begin{refsection}

\section{Introduction}

\subsection{Osteoarthritis}

\subsubsection{Pathology description}

Joint pain continues continues to plague the world's population, affecting an
estimated 27 million Americans alone and encumbering an economic burden of costs
(both direct and indirect) associated with osteoarthritis that exceeds \$60
billion annually.\cite{Brown} As cartilage demonstrates very little tendency for
self-repair, injuries and trauma can further lead to cartilage degeneration and
secondary arthritis, despite the best current care of significant joint
injuries.\cite{Helmerhorst2014} These include injuries to articular cartilage,
anterior cruciate ligament (ACL), and meniscus injuries, contributing to the
accumulated risk of post-traumatic osteoarthritis of
\SIrange{20}{50}{\percent}.\cite{Roos1995,Anderson2011,Reiersen1998,Buckwalter2006}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{f_2_01}
    \caption[Diagram of healthy articular cartilage and that exhibiting
    osteoarthritic pathology]{Diagram of (a) healthy articular cartilage and (b)
        that exhibiting osteoarthritic
        pathology.\cite{Wieland2005}}\label{fig:bone_diagram_1} \end{figure}
% --------------------------
As there is no cure for post-traumatic osteoarthritis, with existing treatments
limited to pain management, the most common outcome is the replacement of the
affected area with joint construction and implantation of a prosthesis. Highly
physically active individuals, such as professional athletes and military
service members, are of particularly higher risk to heightened osteoarthritis
progression.\cite{Cameron2011} Unfortunately, there is a dearth of biological
and chemical compounds available for the treatment of post-traumatic
osteoarthritis. Thus, there exists a potential need for early detection
strategies and therapeutic targets to slow onset and progression.

Cartilage in joints acts as a firm visco-elastic tissue, covering the ends of
bones and acting as a smooth, gliding matrix and as a cushion between bones,
shown in Figure \ref{fig:bone_diagram_1}. It is a biomaterial composed primarily
of collagen fibers and aggrecans (aggregating chondroitin sulphate
proteoglycans), fostered by chondrocyte cells. In healthy cartilage tissue,
articular chondrocytes maintain a stable phenotype, balancing anabolic and
catabolic events, which results in the equilibrium condition of controlled
synthesis and degradation of extracellular matrix components.\cite{Sandell2001}
It is the disturbance of this equilibrium that has been established to be the
mechanism by which osteoarthritis occurs physiologically, resulting in the net
loss of extracellular matrix components key to cartilage function and integrity.
This has been attributed to both the lack of synthesis and increased levels of
proteases, but also has been ascribed to phenotypic changes in articular
chondrocytes, transitioning to a more hypertrophic
phenotype.\cite{Goldring2000,VanderKraan2012} The Kirsch \latin{et al.} have
been amongst the first to report the loss of this phenotype and execution of
hypertrophic and terminal differentiation events,\cite{Kirsch2000} and have been
verified amongst many other research
groups.\cite{Tchetina2007,Merz2003,Pfander2001,Davies2009} 

\subsubsection{Biochemistry of osteoarthritis}

The phenotypic changes of chondrocytes 
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_2_06}
    \caption[]{\cite{Davies2009}}\label{fig:OA_phenotype} \end{figure}
% --------------------------

\subsubsection{The role of retinoids}

Retinoids, and specifically retinoic acids, are arguably the most potent forms
of vitamin A, participating in a broad range of physiological processes
including vision,\cite{Hyatt1996} cell growth and cancer,\cite{Mongan2007}
spermatogenesis,\cite{Vernet2006} inflammation,\cite{Huang2007} and
neural patterning.\cite{Abu-Abed2001}

Retinoids, specifically, \iupac{all-\trans-retinoic acid} (ATRA), and its role
as a regulator of cartilage and skeletal formation has been
established.\cite{Koyama1999} It has further been shown that retinoid signaling
can function in a BMP-independent manner to induce cartilage formation, which
are known to independently stimulate cartilage formation, and furthers the sole
influence of RARs.\cite{Weston2000} Specifically, it has been shown that ATRA is
a very potent inducer of hypertrophic and terminal differentiation, as well as
specifically inducing the expression of certain matrix metalloproteinases (MMPs)
and aggrecanases (ADAMTS), both predominantly responsible for the degradation of
the extracellular matrix at cartilage
sites.\cite{Wang2002,Iwamoto1994,Koyama1999,Johnson2003} These studies have
contributed to the ongoing hypothesis that increased ATRA levels in
osteoarthritis is a major contributor to osteoarthritis pathogenesis because of
ATRA's potential to induce hypertrophic differentiation, inflammatory activity,
increase protease expression, and perturb gene expression associated with
extracellular matrix components. Furthermore, Kirsch \latin{et al.} from New
York University Joint Disease Hospital have preliminary unpublished data
suggesting that ATRA stimulates catabolic events in articular chondrocytes and
further augments these events in the presence of catabolic cytokines, such as
interleukin-1-beta (IL-1${\beta}$).
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_2_07}
    \caption[]{\cite{DeLera2007}}\label{fig:retinoid_receptor_structures} \end{figure}
% --------------------------
The influence of ATRA and its analogues stems from their interactions with two
sub-families of nuclear receptor transcription factors involved in retinoid
signaling - retinoic acid receptors (RARs) and retinoid X receptors (RXRs). RARs
are further differentiated across three variants, RAR${\alpha}$, RAR${\beta}$,
and RAR${\gamma}$ receptors, all three of which are activated by
ATRA.\cite{Chambon1996} While growth plate chondrocytes are known to mainly
express RAR${\alpha}$ and RAR${\gamma}$,\cite{Koyama1999} little is known
regarding the expression of RARs amongst osteoarthritic and healthy articular
chondrocytes. RARs and RXRs interact with one another to form heterodimers,
facilitating ligand-dependent transcription with a myriad of discovered
participating cofactors (Figure \ref{fig:retinoid_receptor_structures}). RXRs, however,
can form ``permissive'' heterodimers with other nuclear receptors, whereby
RXR-selective ligands can activate transcription via the heterodimer on their
own; this is not the case with RAR/RXR heterodimers, viewed as
``non-permissive'', they require the binding of RAR ligands for activation to
occur and otherwise remain silent.\cite{Altucci2007} Activation subsequently
drives physical interactions with coregulatory proteins (i.e. corepressors and
coactivators), ultimately binding to retinoic acid response elements (RAREs)
located at promoter regions of target genes.\cite{Germain2002,DeLera2007}

\subsubsection{Retinoid analogs}

In a comprehensive review of the design of selective RAR and RXR modulators, de
Lera \latin{et al.} elegantly presents the schema for the modulation of RAR-RXR
communication by various ligands. This review aptly summarizes that ``the
allosteric conformational change induced by a ligand onto a nuclear receptor
does nothing but modulate its communication properties with the surrounding
microcosmos of signal adaptors.``\cite{DeLera2007} This applies to the retinoid
superfamliy of receptors as shown in Figure \ref{fig:RAR_communication}<++>
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.95\textwidth]{f_2_08}
    \caption[]{\cite{DeLera2007}}\label{fig:RAR_communication} \end{figure}
%This caption needs to be paraphrased from the DeLera paper.
% --------------------------
Of specific relevance to the work presented herein is the inverse agonist,
BMS493 (\iupac{(\E)4-[2-[5,5-dimethyl
8-(2-phenylethynyl)-5,6-dihydronaphthalen-2-yl]ethen-1-yl]benzoic acid}), which
stabilizes the co-repressor/heterodimer complex, thus leading to enhanced
repression.\cite{Germain2002} 

\subsection{Delivering small molecule therapeutics}

\subsubsection{Hurdles in the delivery of therapeutics}

\subsubsection{Drug delivery vehicles}

In the development of biochemical therapeutics, bioavailability has long been a
defining functional benchmark for many innovations in the field. The bodies
reticuloendothelial system, responsible for the efficient clearance of foreign
small molecules, is unfortunately too efficient at its task at size-excluding
therapeutic small molecules for subsequent excretion via the kidneys.
Bioavailability can be a function of compound solubility, \latin{in vivo}
half-life, therapeutic index, and immunogenicity, which all can affect the
available quantity of compound for its intended
destination.\cite{Petros2010,Davis2008,Shah1992}

Stemming from the long-held small molecule chemistry approach to therapy development,
many of the breakthroughs in the development of molecular ``vehicles'' to
increase bioavailability pertain to small molecules, both hydrophobic and
hydrophilic. Nanoparticle-based drug delivery, in particular, has been
approached from multiple material chemistries, including liposome, polymer, and
protein-based delivery vehicles. Many of these particle-based technologies
leverage the enhanced permeability and retention (EPR) effect, characteristic of
the body's size-exclusive execretory process via its reticuloendothelial
system.\cite{Matsumura1986,Fang2010} The EPR effect is credited to the
physiologic nature of the vasculature of tumor tissue, which tends to be
``leakier'' than healthy tissue, allowing for the mass transfer of larger
particles through the endothelium than would normally be permitted. Furthermore,
the lymphatic drainage of tumor tissue is not capable of draining particles
larger than some permitted through the leaky endothelium, the result of which
is a size cutoff window that allows for accumulation of nanoparticles at a site
of cancerous tissue (Figure \ref{fig:EPR_effect}).
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.95\textwidth]{f_2_04}
    \caption{Schematic of the enhanced permeability and retention (EPR)
    effect.}\label{fig:EPR_effect} \end{figure}

\subsubsection{Mechanisms and strategies for enhancement of delivery}
% --------------------------
Liposomes are poised as the most mature of the technologies, having been
successfully implemented to deliver various therapeutics stemming from their
ability to efficiently sequester small molecule
solutes.\cite{Ranson1996,Sessa1968,Bangham1965,Bangham1974} The potential for
delivery of anti-cancer therapeutics has been pursued since the early
1970's.\cite{Gregoriadis1974} They have particularly been successful in
increasing the bioavailability of anti-cancer therapeutics, such as doxorubicin
and curcumin, whilst having matured to take on forms allowing more advanced
passive and active targeting strategies.\cite{Al-Jamal2012,Huang1994,Li2007}

In addition, polymer-based micelles have also been developed, such as those
consisting of \iupac{poly-(lactic-co-glycolic) acid} (PLGA) for the
encapsulation of small molecules, including curcumin and
\iupac{all-\trans-retinoic acid}.\cite{Li2009,Yallapu2010b} This strategy has
shown to produce drug encapsulation efficiencies of up to \SI{90}{\percent},
dramatically improving the solubility of, for example, a highly water insoluble molecule
such as curcumin. An example of typical formulation strategy is exemplified in
the work by Yallapu \latin{et al.}, shown in Figure
\ref{fig:PLA_curcumin_example}.\cite{Yallapu2010b}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.95\textwidth]{f_2_02}
    \caption[Schematic of typical curcumin encapsulation strategy into PLGA
        co-block polymers.
    ]{Schematic of typical curcumin encapsulation strategy into PLGA co-block
    polymers. A) formulation diagram, combining small molecule with polymeric
encapsulation unit, B) photo suggesting the enhanced solubility of encapsulated
curcumin (middle vial), C) particle size distribution of PLGA-curcumin, D)
transmission electron micrographs of
nanoparticle.\cite{Yallapu2010b}}\label{fig:PLA_curcumin_example} \end{figure}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{f_2_03}
    \caption[Example of improvements in plasma concentrations of ATRA when
    encapsulated in polymeric, PEG-/PLA-based nanoparticles.]{Example of improvements in plasma concentrations of ATRA when
    encapsulated in polymeric, PEG-/PLA-based
    nanoparticles.\cite{Li2009}}\label{fig:PLA_ATRA_example} \end{figure}
% --------------------------
Figure \ref{fig:PLA_ATRA_example} shows the bioavailability results of
ATRA-loaded micelle-like nanoparticles, developed by Nagai \latin{et
al.}\cite{Li2009} improving the half-life of ATRA as compared to an injection of
free ATRA solution.

\subsubsection{Protein-based drug delivery vehicles}

\subsection{Cartilage oligomeric matrix protein}

\subsubsection{Repurposing of coiled-coil domain}

\subsection{Scope of work}

% Document workflow between NYU Joint Disease Hospital and NYU-Poly. This sets
% the pick for small molecule screening.

In collaboration with the group led by Dr. Thorsten Kirsch (NYU Joint Disease
Hospital), we intended to evaluate the ability for a protein construct to bind
to a therapeutic RAR agonist and/or antagonist to demonstrate the feasibility of
non-covalently storing such hydrophobic molecules for long-term delivery. This
work specifically addresses the delivery of the inverse agonist, BMS493 (Figure
\ref{fig:RAR_communication}.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.95\textwidth]{f_2_05}
    \caption[]{\cite{}}\label{fig:research_intention} \end{figure}
% --------------------------
The research goals for this work are the following:
\begin{description}
    \item[Production] Application of high-yield protein biosynthesis methods to
        COMP protein.
    \item[Dosage] Evaluation of the effects of BMS493 and COMP concentration on
        \latin{in vitro} chondrocyte cultures.
        %pivot: endotoxin purification
    \item[Binding] Evaluation of the binding affinity of BMS493 to COMP.
        %pivot: zonal elution non-compliant
    \item[Release] Evaluation of the release of BMS493 from COMP-BMS493
        complexes given sink conditions.
    \item[Formulation] Ensure appropriate concentrations, solubilities, and
        structural stabilities of both BMS493 and protein as a formed complex.
\end{description}

\section{Methods}

\subsection{Recombinant gene construction}

The lazy brown fox.

\subsection{Biosynthesis}

All reagents used for the biosynthesis of COMP protein were sterile filtered
through a \SI{0.22}{\um} filter prior to usage. Biosynthesis of pQE9-COMP was carried out in a chemically competent BL21(DE3)
\emph{E.coli} host cell line grown at large scales with auto-induction media.
The plasmid vector DNA (Figure \ref{fig:plasmid_pQE9-COMP}) was transformed into
the cells following the improved \ch{RbCl2} method of chemical transformation,
specifically abiding by the protocol set forth by Nicholas
Renzette.\cite{Renzette2011} Cells were then plated onto tryptic soy agar plates
(\SI{40}{\mg\per\mL} supplemented with ampicillin (\SI{200}{\ug\per\mL}.
\SI{10}{\mL} starter
cultures were prepared from single colonies, prepared in 1xM9 media and
supplemented with the reagents listed in Table
\ref{tab:COMP_expression_media}.
% --------------------------
%START_TABLE
%AUTO-INDUCTION REAGENTS TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{ ll }
  \hline
  \multicolumn{2}{c}{Component (Final Concentrations)} \\
  \hline

  ampicillin & \SI{200}{\ug\per\mL} \\
  chloramphenicol & \SI{35}{\ug\per\mL} \\
  thiamine hydrochloride & \SI{34}{\ug\per\mL} \\
  magnesium sulfate & \SI{1}{\milli\moLar} \\
  calcium chloride & \SI{0.1}{\milli\moLar} \\
  glucose & \SI{0.4}{\wtper} \\
  20 amino acids & \SI{100}{\ug\per\mL} \\

  \hline
\end{tabular}
\caption{M9 Supplement Specifications}
\label{tab:COMP_expression_media}
\end{table}
%END_TABLE
% --------------------------
Starter cultures were used to inoculate larger scale expression media in either
a baffled flask or a bioreactor format, comprising a \SI{4}{\volper} of the
large scale media. Cells were grown in M9 in a single auto-induction stage as
per the methods set forth by Studier.\cite{Studier2005} As per this method, the
media of the large scale expression was bereft of glucose, as documented in
Table \ref{tab:COMP_expression_media} and instead was supplemented with
\SI{0.5}{\wtper} glycerol, \SI{0.05}{\wtper} glucose, and \SI{0.2}{\wtper}
lactose, the ratios of which have been previously optimized to yield optimally
timed expression of protein controlled by an inducible promoter governed by the
\emph{lac} operon.\cite{Studier2005} Flask expressions were carried out in
\SI{1}{\L} baffled flasks for \SI{8}{\hour} at \SI{37}{\celsius}, shaking at
\SI{350}{\rpm}. In addition, larger scale expression methods were employed by
ways of a \SI{7.5}{\L} BioFlo 110 bioreactor/fermentor (New Brunswick
Scientific), operated in bioreactor mode for \emph{E.coli} growth and equipped
with a water jacketed temperature control system. The reactor was operated with
the parameters shown in Table \ref{tab:bioreactor_parameters}.
% --------------------------
%START_TABLE
%BIOREACTOR PARAMETERS TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{ ll }
  \hline
  \multicolumn{2}{c}{Bioreactor Parameters} \\
  \hline

  Culture volume & \SI{3}{\L} \\
  Temperature & \SI{37}{\celsius} \\
  Flow rate & \SI{3}{\L\per\minute} \\
  Dissolved \ch{O2} & \SI{35}{\percent} \\
  pH & 7.0 \\
  Impeller speed & \SI{450}{\rpm} \\

  \hline
\end{tabular}
\caption{Operating parameters for expression using a BioFlo 110 system}
\label{tab:bioreactor_parameters}
\end{table}
%END_TABLE
The cultures were subsequently havested and pelleted via centrifugation at
\SI{4000}{\gforce} for \SI{20}{\minute} at \SI{4}{\celsius}, and stored
immediately at \SI{-20}{\celsius} for downstream lysis. \SI{1}{\mL} aliquots
of the cultures were sampled at particular time points for downstream SDS-PAGE analysis. Each
aliquot was centrifuged at \SI{12000}{\rpm} prior to decanting of the
supernatant and storage at \SI{-20}{\celsius}.

\subsection{Protein purification}

All solutions used in the extraction and purification of recombinant proteins
were filtered through \SI{0.22}{\um} sterile filters. In addition, all solutions
were adjusted to the appropriate pH with either \SI{37}{\percent} \ch{HCl} or
\SI{1}{\moLar} \ch{NaOH}. Lysis of harvested cells was carried out under
denaturing conditions, assisted with probe sonication. Cell pellets were thawed
at \SI{4}{\celsius} and immediately resuspended in lysis buffer consisting of
\SI{50}{\milli\moLar} \ch{Na2HPO4}, \SI{6}{\moLar} urea, and
\SI{20}{\milli\moLar} imidazole, \pH 8.0. The suspension was then subjected to
sonication with a Q700 probe sonication system (Qsonica), equipped with a
\SI{127}{\mm} probe and operated at \SI{50}{\percent} intensity with a
\SI{50}{\percent} duty cycle across a \SI{10}{\second} period duration,
ultimately transferring approximately \SI{200}{\joule\per\mL} of lysate slurry -
approximately requiring \SI{10}{\minute} for a \SI{100}{\mL} sample, averaging
\SI{33}{\watt}. The lysate was kept secured within an ice bath during sonication
so as to inhibit a significant increase in temperature. The sonicated product
was then clarified of debris by centrifugation at \SI{20000}{\gforce} for
\SI{2}{\hour} at \SI{4}{\celsius}.

The clarified lysate was then applied to a \SI{5}{\mL} HiTrap IMAC FastFlow
column (GE Life Sciences), charged with \ch{CoCl2}, using an \"{A}KTA purifier
system (GE Life Sciences) at \SI{4}{\celsius}. The purifier was equipped with a
sample pump for external loading. The lysate was loaded with
\SI{20}{\milli\moLar} imidazole and washed with 10 column volumes (CV) of the
same buffer constitution. The protein was then eluted across 3 CVs of
\SI{510}{\milli\moLar} imidazole. Purified elutions were assayed for purity via
SDS-PAGE analysis. Polyacrylamide gels were cast using ammonium persulfate and
tetramethylethylenediamine-catalyzed polymerization to obtain \SI{12}{\wtper}
acrylamide gels. Gel lanes of pure protein were used to quantitate
post-purification yields, using an ImageQuant gel imaging system (GE Life
Sciences). Loaded mass of protein, from \SI{10}{\uL}, was calibrated against a
volumetric equivalent of Precision Plus Protein Unstained Standard mixture
(Bio-Rad). Contingent upon confirmation of \SI{90}{\percent} purity, was
desalted using Sephadex G-25 desalting resin, loaded into a pre-packed 5mL
HiTrap column format (GE Life Sciences) and equilibrated with
\SI{10}{\milli\moLar} Gomori buffer, prepared to pH 8.0. Desalting procedure was
accomplished using the \"{A}KTA purifier system. Purification elutions were
loaded into a \SI{10}{\mL} column format superloop for queued \SI{1}{\mL}
injections onto the desalting column. The desalting column was equilibrated with
5 CVs of buffer prior to each injection. Protein samples underwent 2 rounds of
subsequent desalting so as to ensure stringent buffer replacement as well as
accommodating for the relatively high concentrations of imidazole and urea in
the elution fractions.

\subsection{Small molecule characterization}

%\subsubsection{Solubility testing}

\subsubsection{Reverse-phase chromatographic quantitation}



\subsection{Protein complex formulation}

\subsubsection{Solvent condition screening}

The lazy brown fox.

\subsubsection{Circular dichroism spectroscopy}

The lazy brown fox.

\subsubsection{Protein complex separation}

The lazy brown fox.

\subsection{Complex formulation characterization}

\subsubsection{Zonal elution chromatography}

The lazy brown fox.

\subsubsection{Hummel-Dreyer chromatography}

The lazy brown fox.

\subsubsection{Isothermal titration calorimetry}

The lazy brown fox.

\subsection{In vitro analysis}

\subsubsection{Cell culture conditions and sample methods}

The lazy brown fox.

\subsubsection{Small molecule extraction}

The lazy brown fox.

\section{Results}

\subsection{Scaling biosynthetic methods}

\subsection{Screening of agonists/antagonists}

\subsection{Small molecule stability}
%Introduction to light sensitivity of molecules

\subsubsection{Light-induced oxidation}
%DATA: initial chromatography studies 

\subsubsection{\latin{In vitro availability}}
%DATA: extraction method development
%Initial attempt from GM20130716
%Second attempt via SPE GM20130913
%Final attempt, including curves from last MM report.
%DATA: method application to BMS493 dosage study


\subsection{Small molecule binding}

%DATA: zonal elution chromatography
%First attempts with ATRA as a control molecule. Unfortunately no dice, despite
%the promising results, it is probably due to light scattering effects.
%DATA: Hummel Dreyer chromatography

\subsection{Small molecule release}

\section{Discussion}
The lazy brown fox.

\section{Conclusion}
The lazy brown fox.


\printbibliography[heading=subbibliography]

\end{refsection}
