\chapter{Modulation of assembly properties of protein-based biomaterials via
global fluorinated amino acid incorporation}

\begin{refsection}

\section{Introduction}

\subsection{Protein-based materials}

% Introduce proteins as materials - a sort of history survey of bio-based
% functional material and their corresponding various application fields.

Genetically engineered proteins have made a significant impact in biomaterials
research, enabling the fabrication of polymers with monodisperse molecular
weights, a diverse set of side chain functionality, and well-defined secondary
structure elements.\cite{
Yoshikawa1994,Wang2001,Rabotyagova2009,Martin2009,Dinerman2002,Megeed2002,Haider2004,Wright2002,Banta2010,Link2003,Straley2009,Rabotyagova2011}
Compared against a backdrop of predominant synthetic polymers, protein polymers,
particularly those generated via high-fidelity recombinant DNA technologies,
exhibit defined molecular weight, composition, sequence, and
stereochemistry.\cite{} While proteins are not alone amongst engineered
materials of biological origin, they exist as the intersection of biological
function and structure. The latter feature spans the gamut of length scales,
ranging from the nanoscopic to the macroscopic, with notable examples including
actin filaments in the intracellular milieu to the silk fibers visible to the
naked eye.\cite{Rabotyagova2009} Their frequent roles as both sensors and
chemical catalysts are testaments to their functional impact on nature.

% Begin to focus on example of proteins that have secondary structural motifs
% that abstract themselves to higher order structures, such as fibers.
% Generalize how these can span length scales from nanoscopic order to
% macroscopic order.  Why do these higher order structures typically form. 

Numerous examples exist of proteins extrapolating the assembly process beyond
the quaternary scale of folding. This arena of self-assembly is of particular
focus herein, requiring inquiry into the evidence of such phenomenon occurring,
and more importantly, how these self-assembly events occur mechanistically.

In the case of collagen, secondary structure is established as alpha-helices but
further progresses to the self-association of monomers to form triple helices.
More interestingly, this tiered assembly, wherein the larger macromolecular
assemblies become dependent on the secondary structure of monomers imbues the
opportunity for sequence-dependent specificity of assemblies. Indeed this
inherent feature of these self-assembling systems has been leveraged by several
engineering design groups.\cite{}

Similarly, resilin, the proteinaceous material responsible for the elasticity of
various insect wings and appendages, relies on a critical proline residue to
accommodate regular turns in the protein backbone that promote a molecular
spring able to survive innumerable duty cycles.\cite{}

In the case of elastin-based peptides, secondary structure is established as a
preliminary phase of assembly. The prominent theory in the field dictates that
the higher order self-assembly of these class of peptides is governed by the
hydrophobic collapse of the macromolecule. In certain cases, this results in a
macroscopic network of peptides to yield either a gel or colloid of
coacervates.\cite{}

Silk is another prominent natural protein on which novel biomaterial design is
based. It exists as an example of a heterogeneous mixture of structured and
unstructured secondary structures.  $\beta$-sheets dominate the ordered features
of silk variants.

\subsection{Incorporation of non-natural amino acids}

While the physicochemical properties of protein-based materials have been
modulated with extraordinary capacity and range\cite{} - in large part due to
the non-covalent interplay of amino acid residues - they are still limited by
the constrained 22 amino acids that define functional group diversity.\cite{}
This limitation has been addressed, and in many ways deprecated, by various
efforts in genetic engineering.\cite{} 

There exists two contemporary methods to biosynthetically incorporate
non-natural amino acids into proteins: site-specific incorporation and
residue-specific incorporation.

\subsubsection{Site-specific incorporation methods} 

% Document the example of site-specific incorporation of azido-containing amino
% acids into polypeptides.

Site-specific incorporation has been a viable strategy for the incorporation of
non-natural amino acids into polypeptides since the late 1990's. With the
seminal efforts of the Schultz et al., it is now possible to redefine the
genetic code. In 2001, the Schultz group reported on the development of a unique
transfer RNA (tRNA)/aminoacyl-tRNA synthetase pair that were genetically
engineered to exist as orthongal complments to their endogenous analogues, as
well as associate the encoding for the amber nonsense codon with a non-natural
amino acid - O-methyl-L-tyrosine. The group was able to thus incorporate a
non-natural, synthetic amino acid with notably high fidelity
(99\%).\cite{Wang2001} This method relies on the translation suppression of
endogenous amber nonsense codons, which is addressed by the orthogonal nature of
the system's design. Similarly, Kwon and coworkers have been able to reassign
one of the degenerate codons (UUU) that correspond to phenylalanine to encode
for 2-naphthylalanine.\cite{Kwon2003}

Other groups have pushed the envelope still further with this technique,
reimagining the genetic code itself.

\subsubsection{Global incorporation methods} 

% Include an argument in support of residue-specific incorporation via
% endogenous tRNA/aminoacyl-tRNA synthetase

Global incorporation of non-natural amino acids is further made possible by the
particular technique of residue-specific substitution. This technique is
typically characterized by a stochastic substitution of all amino acids of a
particular identity within a given sequence. It is made possible by two
prominent factors: 1) the promiscuity and relaxed specificity of aminoacyl-tRNA
synthetase endogenous to certain expression hosts, and 2) the existence of
auxotrophic expression hosts.

\subsection{Fluorinated amino acids}

% What's the earliest precedence found for fluorinated amino acids and the
% motivation behind their synthesis?


\subsection{Fluorous effect}

The lazy brown fox.

\subsection{Block protein polymers}

\subsubsection{Fusion Proteins}

\subsubsection{Elastin-like peptides}

The lazy brown fox.

\subsubsection{Cartilage oligomeric matrix protein}

The lazy brown fox.

\subsubsection{Block ELP-COMP Protein Polymers}

The lazy brown fox.

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{Schema indicating three block copolymer proteins that were
        previously generated, via recombinant DNA techniques, and incorporated
    with para-fluorophenylalanine via residue-specific incorporation.
    }
    \label{fig:pff_proteins}
\end{figure}

\subsection{Scope of work}

The lazy brown fox.

\section{Methods}

\subsection{Recombinant gene construction}

The lazy brown fox.

\subsection{Biosynthesis}

The lazy brown fox.

\subsection{Protein purification}

The lazy brown fox.

\subsection{Compositional characterization}

\subsubsection{MALDI-ToF mass spectrometry}

The lazy brown fox.

\subsubsection{Amino acid analysis}

The lazy brown fox.

\subsection{Physicochemical characterization}

\subsubsection{UV/Vis spectroscopy}

The lazy brown fox.

\subsubsection{Circular dichroism spectroscopy}

The lazy brown fox.

\subsubsection{Microrheology}

The lazy brown fox.

\section{Results}

\subsection{Biosynthesis of Fluorinated Block Polymers}

The lazy brown fox.

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{}
    \label{fig:biosynthesis_report}
\end{figure}

The lazy brown fox.

\subsection{Secondary Structural Anaylsis}

The lazy brown fox.

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{}
    \label{fig:CD_temp_wl}
\end{figure}

The lazy brown fox.

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{}
    \label{fig:CD_computation}
\end{figure}

The lazy brown fox.

\subsection{Thermoresponsive Supramolecular Assembly}

The lazy brown fox.

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{}
    \label{fig:lcst}
\end{figure}
k
The lazy brown fox.

\subsection{Mechanical Behavior}

The lazy brown fox.

\begin{figure}[h!]
    \centering
    \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{}
    \label{fig:rheology}
\end{figure}

The lazy brown fox.

\section{Discussion}

The lazy brown fox.

\section{Conclusion}

The lazy brown fox.

\printbibliography[heading=subbibliography]

\end{refsection}
