\chapter{Evaluation of the Delivery of RAR Inverse Agonists with a Coiled-coil
Protein}

\begin{refsection}

\section{Introduction}

\subsection{Osteoarthritis}

\subsubsection{Pathology description}

Joint pain continues continues to plague the world's population, affecting an
estimated 27 million Americans alone and encumbering an economic burden of costs
(both direct and indirect) associated with osteoarthritis that exceeds \$60
billion annually.\cite{Brown} As cartilage demonstrates very little tendency for
self-repair, injuries and trauma can further lead to cartilage degeneration and
secondary arthritis, despite the best current care of significant joint
injuries.\cite{Helmerhorst2014} These include injuries to articular cartilage,
anterior cruciate ligament (ACL), and meniscus injuries, contributing to the
accumulated risk of post-traumatic osteoarthritis of
\SIrange{20}{50}{\percent}.\cite{Roos1995,Anderson2011,Reiersen1998,Buckwalter2006}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{f_2_01}
    \caption[Diagram of healthy articular cartilage and that exhibiting
    osteoarthritic pathology]{Diagram of (a) healthy articular cartilage and (b)
        that exhibiting osteoarthritic
        pathology.\cite{Wieland2005}}\label{fig:bone_diagram_1} \end{figure}
% --------------------------
As there is no cure for post-traumatic osteoarthritis, with existing treatments
limited to pain management, the most common outcome is the replacement of the
affected area with joint construction and implantation of a prosthesis. Highly
physically active individuals, such as professional athletes and military
service members, are of particularly higher risk to heightened osteoarthritis
progression.\cite{Cameron2011} Unfortunately, there is a dearth of biological
and chemical compounds available for the treatment of post-traumatic
osteoarthritis. Thus, there exists a potential need for early detection
strategies and therapeutic targets to slow onset and progression.

Cartilage in joints acts as a firm visco-elastic tissue, covering the ends of
bones and acting as a smooth, gliding matrix and as a cushion between bones,
shown in Figure \ref{fig:bone_diagram_1}. It is a biomaterial composed primarily
of collagen fibers and aggrecans (aggregating chondroitin sulphate
proteoglycans), fostered by chondrocyte cells. In healthy cartilage tissue,
articular chondrocytes maintain a stable phenotype, balancing anabolic and
catabolic events, which results in the equilibrium condition of controlled
synthesis and degradation of extracellular matrix components.\cite{Sandell2001}
It is the disturbance of this equilibrium that has been established to be the
mechanism by which osteoarthritis occurs physiologically, resulting in the net
loss of extracellular matrix components key to cartilage function and integrity.
This has been attributed to both the lack of synthesis and increased levels of
proteases, but also has been ascribed to phenotypic changes in articular
chondrocytes, transitioning to a more hypertrophic
phenotype.\cite{Goldring2000,VanderKraan2012} The Kirsch \latin{et al.} have
been amongst the first to report the loss of this phenotype and execution of
hypertrophic and terminal differentiation events,\cite{Kirsch2000} and have been
verified amongst many other research
groups.\cite{Tchetina2007,Merz2003,Pfander2001,Davies2009} Figure
\ref{fig:OA_phenotype} shows the noted up- and down-regulated markers, given the
onset of ATRA-induced osteoarthritis condition.

%\subsubsection{Biochemistry of osteoarthritis}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_2_06}
    \caption[Transciptional effects following treatment of JW0111 cells with
    \iupac{all-\trans-retinoic acid}.]{Transciptional effects following
        treatment of JW0111 cells with \iupac{all-\trans-retinoic acid}.\cite{Davies2009}}\label{fig:OA_phenotype} \end{figure}
% --------------------------

\subsubsection{The role of retinoids}

Retinoids, and specifically retinoic acids, are arguably the most potent forms
of vitamin A, participating in a broad range of physiological processes
including vision,\cite{Hyatt1996} cell growth and cancer,\cite{Mongan2007}
spermatogenesis,\cite{Vernet2006} inflammation,\cite{Huang2007} and
neural patterning.\cite{Abu-Abed2001}

Retinoids, specifically, \iupac{all-\trans-retinoic acid} (ATRA), and its role
as a regulator of cartilage and skeletal formation has been
established.\cite{Koyama1999} It has further been shown that retinoid signaling
can function in a BMP-independent manner to induce cartilage formation, which
are known to independently stimulate cartilage formation, and furthers the sole
influence of RARs.\cite{Weston2000} Specifically, it has been shown that ATRA is
a very potent inducer of hypertrophic and terminal differentiation, as well as
specifically inducing the expression of certain matrix metalloproteinases (MMPs)
and aggrecanases (ADAMTS), both predominantly responsible for the degradation of
the extracellular matrix at cartilage
sites.\cite{Wang2002,Iwamoto1994,Koyama1999,Johnson2003} These studies have
contributed to the ongoing hypothesis that increased ATRA levels in
osteoarthritis is a major contributor to osteoarthritis pathogenesis because of
ATRA's potential to induce hypertrophic differentiation, inflammatory activity,
increase protease expression, and perturb gene expression associated with
extracellular matrix components. Furthermore, Kirsch \latin{et al.} from New
York University Joint Disease Hospital have preliminary unpublished data
suggesting that ATRA stimulates catabolic events in articular chondrocytes and
further augments these events in the presence of catabolic cytokines, such as
interleukin-1-beta (IL-1${\beta}$).
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_2_07}
    \caption[Heterodimeric structure of RAR and RXR nuclear receptors.]{Heterodimeric structure of RAR and RXR nuclear receptors.\cite{DeLera2007}}\label{fig:retinoid_receptor_structures} \end{figure}
% --------------------------
The influence of ATRA and its analogues stems from their interactions with two
sub-families of nuclear receptor transcription factors involved in retinoid
signaling - retinoic acid receptors (RARs) and retinoid X receptors (RXRs). RARs
are further differentiated across three variants, RAR${\alpha}$, RAR${\beta}$,
and RAR${\gamma}$ receptors, all three of which are activated by
ATRA.\cite{Chambon1996} While growth plate chondrocytes are known to mainly
express RAR${\alpha}$ and RAR${\gamma}$,\cite{Koyama1999} little is known
regarding the expression of RARs amongst osteoarthritic and healthy articular
chondrocytes. RARs and RXRs interact with one another to form heterodimers,
facilitating ligand-dependent transcription with a myriad of discovered
participating cofactors (Figure \ref{fig:retinoid_receptor_structures}). RXRs, however,
can form ``permissive'' heterodimers with other nuclear receptors, whereby
RXR-selective ligands can activate transcription via the heterodimer on their
own; this is not the case with RAR/RXR heterodimers, viewed as
``non-permissive'', they require the binding of RAR ligands for activation to
occur and otherwise remain silent.\cite{Altucci2007} Activation subsequently
drives physical interactions with coregulatory proteins (i.e. corepressors and
coactivators), ultimately binding to retinoic acid response elements (RAREs)
located at promoter regions of target genes.\cite{Germain2002,DeLera2007}

\subsubsection{Retinoid analogs}

In a comprehensive review of the design of selective RAR and RXR modulators, de
Lera \latin{et al.} elegantly presents the schema for the modulation of RAR-RXR
communication by various ligands. This review aptly summarizes that ``the
allosteric conformational change induced by a ligand onto a nuclear receptor
does nothing but modulate its communication properties with the surrounding
microcosmos of signal adaptors.``\cite{DeLera2007} This applies to the retinoid
superfamliy of receptors as shown in Figure \ref{fig:RAR_communication}.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{f_2_08}
    \caption[Ligand-driven communication between RAR and RXR
    receptors.]{Ligand-driven communication between RAR and RXR receptors.
        Homeostatic state is indicated by panel (a) wherein the RAR-RXR dimer
        exists with a co-repressor(CoR) bound to the RAR subunit. Upon RAR
        agonist activation, panel (b), the CoR binding interface is disrupted
        and offers a binding site for co-activator (CoA). RXR-selective agonist
        binding promotes the subsequent binding of CoA to sites on both of the
        receptors (c). (d) shows the case of RXR subordination, in which CoA
        cannot bind to the heterodimer. Further in the direction of repression,
        inverse agonists, such as BMS493, can stabilize the heterodimer-CoR
        complex (e). Neutral antagonists generate neither a surface to which CoR or
        CoA may bind. In less common cases, neutral antagonists can allow RXR
        agonists to recruit CoAs, resulting in partial activation.\cite{DeLera2007}}\label{fig:RAR_communication} \end{figure}
%This caption needs to be paraphrased from the DeLera paper.
% --------------------------
Of specific relevance to the work presented herein is the inverse agonist,
BMS493 (\iupac{(\E)4-[2-[5,5-dimethyl
8-(2-phenylethynyl)-5,6-dihydronaphthalen-2-yl]ethen-1-yl]benzoic acid}), which
stabilizes the co-repressor/heterodimer complex, thus leading to enhanced
repression.\cite{Germain2002} 

\subsection{Delivering small molecule therapeutics}

\subsubsection{Hurdles in the delivery of therapeutics}

The primary treatments of osteoarthritis set out to address its symptoms, with
the most common therapeutics being corticosteroids and hyaluronic
acid.\cite{Butoescu2009}. Despite the numerous difficulties associated with
intra-articular injection of therapeutics (e.g. injection inaccuracies,
hyper-reactive inflammatory response of the synovium, and risks of bacterial
infection), the primary reason for failure is related to the rapid efflux of
drugs from the joint cavity.\cite{Ayral2001} This is an obvious result of the
nature of the synovial surface that lines the joint cavity, consisting of a
discontinuous layer of synoviocytes with wide intercellular gaps measuring
\SIrange{0.1}{5.5}{\um} and no basement membrane, and which has been known since
1984.\cite{Knight1984} Due to the nature of the synovial ultrastructure, there
is a rapid flux of water and solutes in a joint cavity, in effect acting very
much like a dialysis chamber; small molecules (MW ${<}$ \SI{10000}{\dalton}) have
been shown to easily diffuse through the interstitium, applying an equilibrium
between the synovial fluid and plasma.\cite{Gerwin2006,Okuyama1984} Even with
hyaluronic acid injections, which comprise MW ranging from
\SIrange{2}{6}{\mega\dalton} and have reported half-lives up to \SI{24}{\hour},
the holy grail still remains obtaining sustained release on the order of days to
weeks.

\subsubsection{Drug delivery vehicles}

In the development of biochemical therapeutics, bioavailability has long been a
defining functional benchmark for many innovations in the field. The bodies
reticuloendothelial system, responsible for the efficient clearance of foreign
small molecules, is unfortunately too efficient at its task at size-excluding
therapeutic small molecules for subsequent excretion via the kidneys.
Bioavailability can be a function of compound solubility, \latin{in vivo}
half-life, therapeutic index, and immunogenicity, which all can affect the
available quantity of compound for its intended
destination.\cite{Petros2010,Davis2008,Shah1992}

Stemming from the long-held small molecule chemistry approach to therapy development,
many of the breakthroughs in the development of molecular ``vehicles'' to
increase bioavailability pertain to small molecules, both hydrophobic and
hydrophilic. Nanoparticle-based drug delivery, in particular, has been
approached from multiple material chemistries, including liposome, polymer, and
protein-based delivery vehicles. Many of these particle-based technologies
leverage the enhanced permeability and retention (EPR) effect, characteristic of
the body's size-exclusive execretory process via its reticuloendothelial
system.\cite{Matsumura1986,Fang2010} The EPR effect is credited to the
physiologic nature of the vasculature of tumor tissue, which tends to be
``leakier'' than healthy tissue, allowing for the mass transfer of larger
particles through the endothelium than would normally be permitted. Furthermore,
the lymphatic drainage of tumor tissue is not capable of draining particles
larger than some permitted through the leaky endothelium, the result of which
is a size cutoff window that allows for accumulation of nanoparticles at a site
of cancerous tissue (Figure \ref{fig:EPR_effect}).
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.95\textwidth]{f_2_04}
    \caption{Schematic of the enhanced permeability and retention (EPR)
    effect.}\label{fig:EPR_effect} \end{figure}

\subsubsection{Mechanisms and strategies for enhancement of delivery}
% --------------------------
Liposomes are poised as the most mature of the technologies, having been
successfully implemented to deliver various therapeutics stemming from their
ability to efficiently sequester small molecule
solutes.\cite{Ranson1996,Sessa1968,Bangham1965,Bangham1974} The potential for
delivery of anti-cancer therapeutics has been pursued since the early
1970's.\cite{Gregoriadis1974} They have particularly been successful in
increasing the bioavailability of anti-cancer therapeutics, such as doxorubicin
and curcumin, whilst having matured to take on forms allowing more advanced
passive and active targeting strategies.\cite{Al-Jamal2012,Huang1994,Li2007}

In addition, polymer-based micelles have also been developed, such as those
consisting of \iupac{poly-(lactic-co-glycolic) acid} (PLGA) for the
encapsulation of small molecules, including curcumin and
\iupac{all-\trans-retinoic acid}.\cite{Li2009,Yallapu2010b} This strategy has
shown to produce drug encapsulation efficiencies of up to \SI{90}{\percent},
dramatically improving the solubility of, for example, a highly water insoluble
molecule such as curcumin. An example of typical formulation strategy is
exemplified in the work by Yallapu \latin{et al.}, shown in Figure
\ref{fig:PLA_curcumin_example}.\cite{Yallapu2010b}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.95\textwidth]{f_2_02}
    \caption[Schematic of typical curcumin encapsulation strategy into PLGA
        co-block polymers.
    ]{Schematic of typical curcumin encapsulation strategy into PLGA co-block
    polymers. A) formulation diagram, combining small molecule with polymeric
encapsulation unit, B) photo suggesting the enhanced solubility of encapsulated
curcumin (middle vial), C) particle size distribution of PLGA-curcumin, D)
transmission electron micrographs of
nanoparticle.\cite{Yallapu2010b}}\label{fig:PLA_curcumin_example} \end{figure}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{f_2_03}
    \caption[Example of improvements in plasma concentrations of ATRA when
    encapsulated in polymeric, PEG-/PLA-based nanoparticles.]{Example of improvements in plasma concentrations of ATRA when
    encapsulated in polymeric, PEG-/PLA-based
    nanoparticles.\cite{Li2009}}\label{fig:PLA_ATRA_example} \end{figure}
% --------------------------
Figure \ref{fig:PLA_ATRA_example} shows the bioavailability results of
ATRA-loaded micelle-like nanoparticles, developed by Nagai \latin{et
al.}\cite{Li2009} improving the half-life of ATRA as compared to an injection of
free ATRA solution.

\subsubsection{Protein-based drug delivery vehicles}

\subsection{Cartilage oligomeric matrix protein}
\footnote{This section was previously presented in Chapter 1 of this
document.}Cartilage oligomeric matrix protein (COMP) is a non-collagenous matrix
protein found in the cartilaginous extracellular matrix, discovered in the early
1990's, first isolated and characterized in a native form from Swarm rat
chondrosarcoma.\cite{Morgelin1992} It is a member of the thrombospondin (TSP)
gene family of extracellular glycoproteins, found mainly in articular cartilage,
tendon, and
ligaments.\cite{Adams2001,Smith1997,Muller1998,Hedbom1992,Oldberg1992} Early
SDS-PAGE and electron microscopy studies indicated the ability of COMP to
pentamerize (Figure \ref{fig:COMP_EM}).\cite{DiCesare1995,Morgelin1992} 
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{f_2_09}
    \caption[COMP revealed by glycerol spraying and rotary shadowing with
    electron microscopy.]{COMP revealed by glycerol spraying and rotary
        shadowing with electron microscopy. Micrographs sampled from the 1995
        work by DeCesare \latin{et al.} clearly displays the five-armed
        molecules of COMP.\cite{DiCesare1995}}\label{fig:COMP_EM} \end{figure}
% --------------------------
The structure of the coiled-coil domain received heightened attention in the mid
to late 1990's for its channel-like structure. Work by Efimov and Malashkevich
revealed the dimensions of the coiled-coil domain, as well as hydrophobic
channel situated at the center of the pentameric ${\alpha}$-helical
bundle.\cite{Efimov1996,Malashkevich1996a} Structurally, the ${\alpha}$-helical
domain forms a homopentameric, parallel, left-handed coiled-coil with an average
length of \SI{70}{\angstrom} and an average outer diameter of approximately
\SI{30}{\angstrom}. The sequence obeys a conventional ${\alpha}$-helical heptad
motif pattern, with \emph{a} and \emph{d} positions predominantly occupied by
aliphatic, non-polar residues.\cite{Burkhard2001} 
% --------------------------
\begin{figure}
    \centering
    \begin{subfigure}[b]{0.45\textwidth}
        \includegraphics[width=\textwidth]{f_2_10a} \caption[Dimensions of
        coiled-coil COMP domain, shown to bind small hydrophobic molecules such
    as two vitamin \ch{D3} molecules here.]{Dimensions of coiled-coil COMP
        domain, shown to bind small hydrophobic molecules such as two vitamin
        \ch{D3} molecules here.\cite{Ozbek2002}}
        \label{fig:COMPcc_dimensions}
    \end{subfigure}
    \begin{subfigure}[b]{0.45\textwidth}
        \includegraphics[width=\textwidth]{f_2_10b}
        \caption[Comparison of the coiled-coil domain structures of COMP and the
        right-handed coiled-coiled (RHCC).]{Comparison of the coiled-coil domain structures of COMP and the
            right-handed coiled-coiled
            (RHCC).\cite{McFarlane2012,Efimov1996,Malashkevich1996a,Stetefeld2000}}
        \label{fig:coiled-coil_comparison}
    \end{subfigure}
    \caption{Models of the two primary candidates for
        coiled-coil-based delivery agents.}\label{fig:coiled-coil_models}
\end{figure}
% --------------------------
This later inspired work by Ozbek and Guo to study the putative role for COMP as
a storage and delivery protein for regulatory molecules in bone metabolism,
complexing with known transcriptional cofactors such as vitamin \ch{D3},
\iupac{all-\trans retinol}, \iupac{all-\trans retinoic acid}, and
benzene.\cite{Guo1998,Ozbek2002} Even still, interest in the binding
capabilities of coiled-coil domain of COMP has recently reemerged in the groups
of Stetefeld and Montclare, who have published data on the affinity of COMP for
fatty acids and curcumin, respectively, arguably with more intent on engineering
the COMP coiled-coil domain as an intentional
complexer.\cite{McFarlane2012,Gunasekar2009} These as well as the Montclare
group have shown the COMP coiled-coil to demonstrate dissociation constants in
the
\SIrange[scientific-notation=true,retain-unity-mantissa=false]{1e2}{1e-1}{\micro\moLar}
range.\cite{Haghpanah2010,Guo1998}

\subsubsection{Repurposing of coiled-coil domain}

The field of chemical biology has experienced an increase in interest in the
potential of naturally-derived coiled-coil proteins and peptides to serve as
functionally engineered macromolecules. Much of the work, as is evidenced by the
efforts devoted to the COMP coiled-coil, is devoted to probing these proteins
beyond the endogenous roles, and observing emergent properties, both functional
and structural. 

Already, Gunasekar and Montclare have shown that it is possible to tune, within
a degree of predictability, the binding affinities of coiled-coils to small
molecules.\cite{Gunasekar2009} Indeed, \latin{de novo} efforts are also underway
to develop coiled-coils for medical imaging applications, the development of a
binuclear metal binding motif, as well as the creation of a comprehensive basis
set of peptides with which to aid in the design of \latin{de novo}
coiled-coils.\cite{Shiga2012,Fletcher2012,Berwick2014}

And it is in this vein of protein engineering - which has matured to the point
of discrete control of sequence, structure, and function - that the work herein
intends to operate. With the ability to form macroscopic fiber-like assemblies
whilst encapsulating small hydrophobic molecules in its pore region with
\si{\micro\moLar} to \si{\nano\moLar} affinities,
the cartilage oligomeric matrix protein documented herein (Figure
\ref{fig:our_COMP}) may provide the necessary behavior to sustain therapeutics
in the synovial joint cavity for extended periods.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_2_11}
    \caption[COMP coiled-coil primary sequence]{COMP coiled-coil primary sequence\cite{Gunasekar2009}}\label{fig:our_COMP} \end{figure}
% --------------------------

\subsection{Scope of work}

% Document workflow between NYU Joint Disease Hospital and NYU-Poly. This sets
% the pick for small molecule screening.

In collaboration with the group led by Dr. Thorsten Kirsch (NYU Joint Disease
Hospital), we intended to evaluate the ability for a protein construct to bind
to a therapeutic RAR agonist and/or antagonist to demonstrate the feasibility of
non-covalently storing such hydrophobic molecules for long-term delivery. This
work specifically addresses the delivery of the inverse agonist, BMS493 (Figure
\ref{fig:RAR_communication}. In taking a protein engineering approach to the
problem of bioavailability, we intend to evaluate the efficacy of the
coiled-coil domain of rat cartilage oligomeric matrix protein at binding to
BMS493 and correlate these results to concurrently undertaken \latin{in vitro}
studies of the effects of protein-complexed BMS493 on its therapeutic efficacy,
led by the Kirsch Group. From this point on, the coiled-coil domain with which
this work was conducted will be referred to as simply COMPcc.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.95\textwidth]{f_2_05}
    \caption[]{\cite{}}\label{fig:research_intention} \end{figure}
% --------------------------
The research goals for this work are the following:
\begin{description}
    \item[Production] Application of high-yield protein biosynthesis methods to
        COMP protein.
    \item[Dosage] Evaluation of the effects of BMS493 and COMP concentration on
        \latin{in vitro} chondrocyte cultures.
        %pivot: endotoxin purification
    \item[Binding] Evaluation of the binding affinity of BMS493 to COMP.
        %pivot: zonal elution non-compliant
    \item[Release] Evaluation of the release of BMS493 from COMP-BMS493
        complexes given sink conditions.
    \item[Formulation] Ensure appropriate concentrations, solubilities, and
        structural stabilities of both BMS493 and protein as a formed complex.
\end{description}
% --------------------------
In anticipation for the need of a large quantity of protein mass for the
effective distribution in either \latin{in vitro} cultures and/or 
osteoarthritic murine models, partial effort was devoted to migrating the
biosynthesis of COMPcc to a high-yield, auto-induction expression workflow. This
was accomplished by through efforts, namely, the adoption of \ch{RbCl2}
chemically competent expression hosts, optimization of auto-induction media
composition, and the scaling of these methods to a bioreactor format. The
results of benchmark observations pertaining to these efforts are in the Results
section.

Quantification of BMS493 from cultures required the optimization of an
extraction methodology, two of which were investigated.\footnote{Solid phase
    extraction methods, using commercially available Sep-Pak \ch{C18} and Oasis
    gel columns (both Waters Corporation), were also evaluated, but the data is
not presented in this particular draft of this document.} It was deemed critical
to attain high extraction efficiencies of BMS493, as well as ATRA, from aqueous
media so as to properly quantify \si{\nano\mol} amounts of analyte,
indiscernible via conventional spectroscopic instrumentation. This would aid in
not only evaluating the optimal dosage of BMS493, but also offer insight as to
whether the BMS493 is taken up by cells, released, degraded, or residing in the
culture medium. Furthermore, available proven conditions for HPLC analysis of, at
the very least, ATRA and its isomers exist in the literature with which a basis
was built upon for BMS493
separation.\cite{Kim2010b,Chauveau-Duriot2010,Schaffer2010a,Kane2008b,DeLeenheer1982,Motto1989} 

The evaluation of binding affinities of could be approached from numerous
biophysical aspects, including intristic fluorescence assays, cooperative
fluorescence of the small molecule upon binding, and cross-linking studies.
However, the first two techniques do not lend themselves to the presented
system, which does not entertain a fluorescently-active chromophore, or an
environmentally shift-worthy tryptophan residue; the later option does not lend
itself to conclusive nor high-throughput adaption. As such, two methods were
investigated for the evaluation of binding affinities: zonal elution
chromatography and Hummel-Dreyer chromatography.

Zonal elution chromatography involves the elution of a
macromolecule/small-molecule complex from free small-molecule via size exclusion
chromatographic separations, like those attained with with G-25 Sephadex resin
media, which segregates based on a narrow molecular weight cutoff ( ${<}$
\SI{1000}{\dalton}). This method is particularly suited to irreversible binding
conditions, such as the loading of liposomes and subsequent separation from
remaining free small molecule payload.\cite{Pan2012,Wang2013} If, however, the
mechanism of loading of BMS493 into COMPcc occurs as to achieve equilibrium,
zonal elution will prove invalid. Because of the existence of axial dispersion
and resolution into separate zones, both characteristics of a gel filtration
column such as G-25 Sephadex, this method does not hold up to reversible complex
formation; the dilution gives rise to an ever decreasing propoertion of complex,
the concentration of which tends to zero as the resolution of the two reactants
is effected.\cite{Winzor2001} Yet, since it is not know by which mode the BMS493
will bind to the COMPcc protein, both methods will be attempted.
 
Hummel Dreyer chromatography is a variant of zonal elution chromatography, but
is able to withstand the rapid equilibrating interaction occurring at the fluid flow
front. This method involves the application of a small zone of acceptor protein
to a column of tightly cross-linked gel pre-equilibrated with a known
concentration of ligand.\cite{Hummel1962} The resultant elution profile exhibits
a peak of ligand concentration coincident with elution of acceptor as the
results of complex formation during acceptor zone migration through a
concentration of free ligand. This method has been adapted successfully to both
FPLC and HPLC formats.\cite{Bieri1998}\footnote{The results for Hummel Dreyer
    analysis are still being collected and are thus not present in this draft of
the document.}

\section{Methods}

\subsection{Protein preparation methods}

\subsubsection{Biosynthesis}

All reagents used for the biosynthesis of COMP protein were sterile filtered
through a \SI{0.22}{\um} filter prior to usage. Biosynthesis of pQE9-COMP was
carried out in a chemically competent BL21(DE3) \emph{E.coli} host cell line
grown at large scales with auto-induction media.  The COMP gene in the pQE9
vector (Figure \ref{fig:plasmid_pQE9-COMP}), a gift from Zhang,\cite{Shen2006a}
was transformed into the cells following the improved \ch{RbCl2} method of
chemical transformation, specifically abiding by the protocol set forth by
Nicholas Renzette.\cite{Renzette2011} Cells were then plated onto tryptic soy
agar plates (\SI{40}{\mg\per\mL} supplemented with ampicillin
(\SI{200}{\ug\per\mL}.  \SI{10}{\mL} starter cultures were prepared from single
colonies, prepared in 1xM9 media and supplemented with the reagents listed in
Table \ref{tab:COMP_expression_media}.
% --------------------------
%START_TABLE
%AUTO-INDUCTION REAGENTS TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{ ll }
  \hline
  \multicolumn{2}{c}{Component (Final Concentrations)} \\
  \hline

  ampicillin & \SI{200}{\ug\per\mL} \\
  chloramphenicol & \SI{35}{\ug\per\mL} \\
  thiamine hydrochloride & \SI{34}{\ug\per\mL} \\
  magnesium sulfate & \SI{1}{\milli\moLar} \\
  calcium chloride & \SI{0.1}{\milli\moLar} \\
  glucose & \SI{0.4}{\wtper} \\
  20 amino acids & \SI{100}{\ug\per\mL} \\

  \hline
\end{tabular}
\caption{M9 Supplement Specifications}
\label{tab:COMP_expression_media}
\end{table}
%END_TABLE
% --------------------------
Starter cultures were used to inoculate larger scale expression media in either
a baffled flask or a bioreactor format, comprising a \SI{4}{\volper} of the
large scale media. Cells were grown in M9 in a single auto-induction stage as
per the methods set forth by Studier.\cite{Studier2005} As per this method, the
media of the large scale expression was bereft of glucose, as documented in
Table \ref{tab:COMP_expression_media} and instead was supplemented with
\SI{0.5}{\wtper} glycerol, \SI{0.05}{\wtper} glucose, and \SI{0.2}{\wtper}
lactose, the ratios of which have been previously optimized to yield optimally
timed expression of protein controlled by an inducible promoter governed by the
\emph{lac} operon.\cite{Studier2005} Flask expressions were carried out in
\SI{1}{\L} baffled flasks for \SI{8}{\hour} at \SI{37}{\celsius}, shaking at
\SI{350}{\rpm}. In addition, larger scale expression methods were employed by
ways of a \SI{7.5}{\L} BioFlo 110 bioreactor/fermentor (New Brunswick
Scientific), operated in bioreactor mode for \emph{E.coli} growth and equipped
with a water jacketed temperature control system. The reactor was operated with
the parameters shown in Table \ref{tab:bioreactor_parameters}.
% --------------------------
%START_TABLE
%BIOREACTOR PARAMETERS TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{ ll }
  \hline
  \multicolumn{2}{c}{Bioreactor Parameters} \\
  \hline

  Culture volume & \SI{3}{\L} \\
  Temperature & \SI{37}{\celsius} \\
  Flow rate & \SI{3}{\L\per\minute} \\
  Dissolved \ch{O2} & \SI{35}{\percent} \\
  pH & 7.0 \\
  Impeller speed & \SI{450}{\rpm} \\

  \hline
\end{tabular}
\caption{Operating parameters for expression using a BioFlo 110 system}
\label{tab:bioreactor_parameters}
\end{table}
%END_TABLE
The cultures were subsequently havested and pelleted via centrifugation at
\SI{4000}{\gforce} for \SI{20}{\minute} at \SI{4}{\celsius}, and stored
immediately at \SI{-20}{\celsius} for downstream lysis. \SI{1}{\mL} aliquots of
the cultures were sampled at particular time points for downstream SDS-PAGE
analysis. Each aliquot was centrifuged at \SI{12000}{\rpm} prior to decanting of
the supernatant and storage at \SI{-20}{\celsius}.

\subsubsection{Protein purification}
All solutions used in the extraction and purification of recombinant proteins
were filtered through \SI{0.22}{\um} sterile filters. In addition, all solutions
were adjusted to the appropriate pH with either \SI{37}{\percent} \ch{HCl} or
\SI{1}{\moLar} \ch{NaOH}. Lysis of harvested cells was carried out under
denaturing conditions, assisted with probe sonication. Cell pellets were thawed
at \SI{4}{\celsius} and immediately resuspended in lysis buffer consisting of
\SI{50}{\milli\moLar} \ch{Na2HPO4}, \SI{6}{\moLar} urea, and
\SI{20}{\milli\moLar} imidazole, \pH 8.0. The suspension was then subjected to
sonication with a Q700 probe sonication system (Qsonica), equipped with a
\SI{127}{\mm} probe and operated at \SI{50}{\percent} intensity with a
\SI{50}{\percent} duty cycle across a \SI{10}{\second} period duration,
ultimately transferring approximately \SI{200}{\joule\per\mL} of lysate slurry -
approximately requiring \SI{10}{\minute} for a \SI{100}{\mL} sample, averaging
\SI{33}{\watt}. The lysate was kept secured within an ice bath during sonication
so as to inhibit a significant increase in temperature. The sonicated product
was then clarified of debris by centrifugation at \SI{20000}{\gforce} for
\SI{2}{\hour} at \SI{4}{\celsius}.

The clarified lysate was then applied to a \SI{5}{\mL} HiTrap IMAC FastFlow
column (GE Life Sciences), charged with \ch{CoCl2}, using an \"{A}KTA purifier
system (GE Life Sciences) at \SI{4}{\celsius}. The purifier was equipped with a
sample pump for external loading. The lysate was loaded with
\SI{20}{\milli\moLar} imidazole and washed with 10 column volumes (CV) of the
same buffer constitution. The protein was then eluted across 3 CVs of
\SI{510}{\milli\moLar} imidazole. Purified elutions were assayed for purity via
SDS-PAGE analysis, with an expected molecular weight of \SI{6.9}{\kilo\dalton}.
Polyacrylamide gels were cast using ammonium persulfate and
tetramethylethylenediamine-catalyzed polymerization to obtain \SI{12}{\wtper}
acrylamide gels. Gel lanes of pure protein were used to quantitate
post-purification yields, using an ImageQuant gel imaging system (GE Life
Sciences). Loaded mass of protein, from \SI{10}{\uL}, was calibrated against a
volumetric equivalent of Precision Plus Protein Unstained Standard mixture
(Bio-Rad). Contingent upon confirmation of \SI{90}{\percent} purity, was
desalted using Sephadex G-25 desalting resin, loaded into a pre-packed 5mL
HiTrap column format (GE Life Sciences) and equilibrated with
\SI{100}{\milli\moLar} Gomori buffer, prepared to pH 8.0. Desalting procedure
was accomplished using the \"{A}KTA purifier system. Purification elutions were
loaded into a \SI{10}{\mL} column format superloop for queued \SI{1}{\mL}
injections onto the desalting column. The desalting column was equilibrated with
5 CVs of buffer prior to each injection.

\subsubsection{Endotoxin removal}
Protein elutions were separated from pyrogenic lipopolysaccharides using
a \SI{1}{\mL} gravity column loaded with Detoxi-Gel (Pierce) consisting of
Polymixin B, an antibiotic that contains a cationic cyclopeptide with a fatty
acid chain, immobilized onto a network of \SI{6}{\percent} cross-linked beaded
agarose. All solutions (except for sample) were prepared with pyrogen-free water
generated with an Integral 3 MilliQ water purification system (EMD Millipore),
equipped with a UV lamp submersed in the water storage tank to inhibit bacterial
growth and biofilm proliferation as well as a point-of-use \SI{0.22}{\um}
Bio-Pak filter, for the generation of pyrogen-free, DNAase-free, and RNase-free
water. All solutions were additionally filtered through sterile \SI{0.22}{\um}
non-pyrogenic syringe filters prior to immediate usage with the column and
degassed under high vacuum (\SI{15}{\um}) for \SI{20}{\minute}. The
column was regenerated prior and immediately after each challenge with protein
sample by equilibrating it with \SI{5}{\mL} of \SI{1}{\wtper} deoxycholate (in
the form of deoxycholic acid, sodium salt). This was then followed by washing
with \SI{5}{\mL} of pyrogen-free water. The column was then equilibrated with
\SI{5}{\mL} of \SI{10}{\milli\moLar} Gomori phosphate buffer,
\SI{500}{\milli\moLar} \ch{NaCl} prepared to pH
8.0. Just prior to the liquid level surpassing the top frit, the sample was
loaded onto the column. As the sample completely infused the bed of resin,
beyond the frit, the column was sealed at both ends and the sample was allowed
to incubate in the column for \SI{1}{\hour}. The column was the loaded with
additional buffer and was allowed to elute the sample.

\subsubsection{Protein concentration}
Purified protein product was assayed for concentration by way of a Thermo
Scientific Micro BCA Protein Assay Kit, which utilizes bicinchoninic acid as the
detection reagent for \ch{Cu^1+}, which is formed when \ch{Cu^2+} is reduced by
protein in an alkaline environment.\cite{Smith1985}

\subsection{In vitro methods}

\subsubsection{Cell culture methods}
Articular chondrocytes were isolated from 2-month-old wild-type mice.  Cells
were grown in monolayer cultures in Dulbecco's Modified Eagle (DME; Lonza)
medium containing \SI{5}{\percent} fetal calf serum, \SI{2}{\milli\moLar}
\iupac{\L-glutamine} and \SI{50}{U\per\mL} penicillin and streptomycin. After
the cells reached confluency, they were then cultured in the presence of ATRA in
a concentration of \SI{200}{\nano\moLar}, which is approximately the
concentration observed in the synovial fluid of osteoarthritic
patients,\cite{Davies2009} and also observed to be most effective in stimulating
the mRNA levels of catabolic and hypertrophic markers in human and mouse
articular chondrocytes.\cite{Campbell2013b} Cells were grown in the presence of
ATRA for 24 hours at which point the culture media was supplemented with
COMP -/+ BMS493.

The COMP + BMS493 was prepared by combining \SI{125}{\micro\moLar} COMP protein
with an ethanol solution of BMS493 prepared to \SI{500}{\micro\moLar} resulting
in a final concentration of \SI{100}{\micro\moLar} COMP, \SI{100}{\micro\moLar}
BMS493 and \SI{20}{\volper} ethanol. The formulation was incubated at
\SI{4}{\celsius} for 24 hours prior to supplementation. The mixture was added to
cell culture media as a 1:100 diluted supplement, yielding final concentrations
of \SI{1}{\micro\moLar} COMP and BMS493.

\subsubsection{Reverse transcription-polymerase chain reaction (PCR) and
real-time PCR analysis} RNA was isolated from cell cultures using an RNeasy Mini
kit (Qiagen). Levels of messenger RNA (mRNA) for COX-2, IL-6, iNOS, MMP-13, and
type X collagen were quantified by real-time PCR. \SI{1}{\micro\gram} RNA was
reverse transcribed using Omniscript RT kit (Qiagen). The resultant cDNA was
used as the template to quantitate the relative content of mRNA using a 1:100
dilution, analyzed with a ABI Prism 7300 sequence detection system) using
respective primers and SYBR Green.  PCR reactions were performed with a SYBR
Green PCR Master Mix kit (Applied Biosystems) and the primers listed in Table
\ref{tab:pcr_primers} at \SI{95}{\celsius} for 10 minutes, followed by 40 cycles
of \SI{95}{\celsius} for 15 seconds and \SI{60}{\celsius} for 1 minute. 18S RNA
was amplified at the same time and used as an internal control. Transcript
levels were calculated according to the equation ${x=s^{- \Delta C q}}$, where
${\Delta C q = C q_{exp} - C q _{18S}}$.

%START_TABLE
%RT PCR Primers TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{ ll }
  \hline
  \multicolumn{2}{c}{Real-time PCR Primers} \\
  \hline

  COX-2 & forward primer 5'-CATGCCTATCTGCTCTTCGTT-3' \\
  & reverse primer 5'-AGCCACCACACCGTCCATAA-3' \\
  IL-6 & forward primer 5'-AGAGACATCAGGCCATTTTCAGA-3' \\
  & reverse primer 5'-CTGCCATCAGGATCTCTATTTGC-3' \\
  iNOS & forward primer 5'-GCGGCTGATTGTAAGCTTGAT-3' \\
  & reverse primer 5'-GTCGGTGGTCCAGCACTTA-3' \\
  MMP-13 & forward primer 5'-GGAGCTATGATGACACTAACCATGT-3' \\
  & reverse primer 5'-CCAATTCCTGGGAAGACCTCTT Probe-3' \\
  type X collagen & forward primer 5'-AGTGCTGTCATTGATCTCATGGA-3' \\
  & reverse primer 5'-TCAGAGGAATAGAGACCATTGGA \\

  \hline
\end{tabular}
\caption{PCR primers used for real-time PCR analysis. PCR primers were designed
using Primers Express software.}
\label{tab:pcr_primers}
\end{table}
%END_TABLE

\subsection{Analytical methods}

\subsubsection{Small molecule extraction}

%An dosage study of BMS493 was carried out across the following concentrations:
%0, 0.01, 0.1, 0.5, 1, 10, and \SI{20}{\micro\moLar}.  \SI{50}{\uL} aliquots were
%retrieved from the initial \SI{2}{\mL} cultures at specific time points: 0, 1,
%6, 24, and \SI{30}{\hour}. Each of the aliquots was then subjected to an
%extraction technique for subsequent quantitation via UPLC for BMS493 content.
%Cultures were thereafter supplemented with \SI{1}{\micro\moLar} BMS493 as a
%treatment method.

Two variations of liquid-liquid extraction were employed to assist in the
quantitation of BMS493 in culture samples. The first method consisted of
extracting hydrophobic molecules, including BMS493, using ethyl acetate as an
organic phase to be layered atop a 1:1 volumetric equivalent of
\SI{100}{\milli\moLar} Gomori phosphate buffer, pH 8.0, \SI{1}{\percent} DMSO,
\SI{10}{\micro\moLar} analyte. For preliminary feasibility studies, ATRA was
used as an analyte due to its relatively higher absorption coefficient and
increased water solubility. \SI{500}{\uL} of ethylacetate was mixed with
\SI{500}{\uL} of sample and vortexed vigorously for 2 minutes. \SI{400}{\uL} of
the organic phase was then sampled and dried using a vacuum centrifuge
(Labconco). The dried extract was subsequently dissolved in ethanol analyzed
for an absorption spectrum.

The second method of liquid-liquid extraction employed iterated on the technique
listed in the first method. Extraction was carried out across 5 replicates of
controls, consisting of either MilliQ water or DME medium. Each replicate was
spiked with \SI{1}{\milli\moLar} BMS493, prepared in DMSO, to yield final
concentrations of \SI{1}{\micro\moLar} BMS493, \SI{1}{\percent} DMSO.
\SI{50}{\uL} was transferred from these samples to a borosilicate glass test
tube and mixed with \SI{1}{\mL} of spectroscopic grade ethyl acetate (20:1
volumetric ratio). The organic/aqueous sample was then agitated for
\SI{30}{\minute} in a sonicator bath, which had been pre-chilled to
approximately \SI{10}{\celsius}. The samples were then centrifuged for
\SI{5}{\minute} to separate the phases again. \SI{900}{\uL} of the organic phase
was then transferred to a separate glass test tube for vacuum
centrifuge-assisted drying.  The vacuum centrifuge was operated at
\SI{60}{\celsius} for \SI{60}{\minute}, equipped with a dry ice trap to
accommodate the  ethyl acetate condensation. The dried products were then
incubated at \SI{-80}{\celsius} and subsequently freeze-dried using a Freeze
Zone 6 flask lyophilizer unit (Labconco), which was allowed to take place
overnight. This final step was necessary to sublime the trace amounts of DMSO
that had transferred to the ethyl acetate phase. The bone dry product was then
dissolved in \SI{100}{\uL} of ethanol or acetonitrile and injected onto a
\ch{C18} reverse-phase column for quantitation against a set of standards.

BMS493 standards were prepared from dry BMS493 (Sigma Aldrich) that was
resuspended in DMSO to yield a concentration of \SI{4}{\mg\per\mL}.
\SI{50.55}{\uL} of the solution was then freeze-dried and dissolved in
\SI{1}{\mL} of ethanol or acetonitrile to yield a \SI{500}{\micro\moLar} stock
solution. This was then used to make diluted amounts of 5, 2.5, 1, 0.75, 0.5,
0.25, and \SI{0.1}{\micro\moLar} solutions. Each standard was spiked with retinyl
palmitate as an internal standard, set constant at \SI{10}{\micro\moLar}.
Samples and standards were then analyzed using an Acquity H-Class UPLC system,
as per the aforementioned methods.

\subsubsection{Reverse-phase chromatographic quantitation}

Reverse-phase chromatography was carried out on a Acquity H-Class ultra-high
pressure/performance liquid chromatography (UPLC) system (Waters Corporation),
equipped with a tunable UV detector, thermostatted sample compartment, and
column heater. For all retinoid and retinoid agonists, the sample compartment
was kept at \SI{4}{\celsius}. An Acquity UPLC Ethylene Bridged Hybrid (BEH) \ch{C18} column,
\SI{130}{\angstrom}, \SI{1.7}{\um}, \SI{2.1}{\mm} ${\times}$ \SI{100}{\mm}
column (Waters Corporation) was used for separations, equipped with a VanGuard
\ch{C18} pre-column (Waters Corporation). The system was configured with a
tertiary solvent system consisting of MilliQ water, acetonitrile, and methanol,
all at \SI{0.1}{\percent} formic acid. Specific gradients varied amongst methods
on the path toward optimization of protocols, and are listed in the
chromatograms. The pump wash solvent consisted of \SI{10}{\percent}
acetonitrile. The sample manager's needle wash solvent and syringe purge solvent
consisted of acetonitrile (\SI{0.1}{\percent} formic) and \SI{70}{\percent}
acetonitrile, respectively. The column was kept at \SI{40}{\celsius}. All runs
were performed at \SI{1.17}{\mL\per\minute} unless otherwise stated. The TUV
detector was configured to collect at \SI{324}{\nm}, \SI{329}{\nm}, and
\SI{354}{\nm}, in addition to a low-sensitivity 2D spectral scan.

\subsection{Biophysical methods}

\subsubsection{Circular dichroism spectroscopy}

Circular dichroism was carried out consistent with previously reported
methods.\cite{Haghpanah2009,Gunasekar2009,Haghpanah2010,Yuvienco2012} Briefly,
circular dichroism spectroscopy was carried out on a J-815 CD polarimeter
(Jasco) equipped with a PTC-423S single position Peltier temperature control
system and cuvette holder, counter-cooled with an Isotemp 3016S water bath
(Fisher Scientific) set to \SI{22}{\celsius}. Sample solutions were inserted
into a Helma 218 quartz cuvette (\SI{300}{\uL} capacity, \SI{1}{\mm} path
length). Wavelength scans were performed with a \SI{1}{\nm} step size, collected
across 3 consecutive scans.

%\subsubsection{Protein complex separation}

%\subsection{Complex formulation characterization}

\subsubsection{Zonal elution chromatography}

Zonal elution chromatography was accomplished using Sephadex G-25 size-exclusion
resin pre-packed into a \SI{5}{\mL} HiTrap column format (GE Life Sciences). The
column was attached to a \"{A}KTA purifier system (GE Life Sciences) held at
\SI{4}{\celsius}. Protein and/or small molecule samples were prepared in
\SI{10}{\milli\moLar} Gomori phosphate buffer, prepared to pH 8.0 and a range
volumetric proportions of organic co-solvent to facilitate solvation of the
hydrophobic retinoid anologues. The system was configured with one buffer line
inserted into the sample sample buffer composition, filtered, and immediately
degassed via sonication whilst under vacuum immediately prior to usage. Up to
\SI{10}{\mL} of sample was loaded into a \SI{10}{\mL} column format superloop so
as to limit the dilution effects of the injection process. A method was designed
to periodically inject \SI{1}{\mL} of sample, to yield a time-course
investigation of the protein complex separation process.

\subsubsection{Hummel-Dreyer chromatography}

Hummel-Dreyer chromatography was carried out on an Acuity H-Class UPLC system
(Waters Corporation) equipped with a BEH125 SEC column, \SI{1.7}{\um},
\SI{4.6}{\mm} ${\times}$ \SI{300}{\mm}. The pump seal wash consisted of
\SI{10}{\percent} methanol. The isocratic mobile phase was composed via a binary
solvent system consisting of \SI{10}{\milli\moLar} Gomori potassium phosphate
buffer, prepared to pH 7.4, \SI{10}{\volper} methanol, containing either
\SI{0}{\micro\moLar} or \SI{10}{\micro\moLar} BMS493, prepared from a
\SI{100}{\micro\moLar} stock of BMS493 prepared in methanol, which is added to
buffer as \SI{10}{\volper}. Isocratic mobile phases of varying concentrations of
BMS493 were composed via binary combinations of the two solvent lines. All runs
were performed at \SI{0.2}{\mL\per\minute} unless otherwise stated.

\section{Results}

\subsection{Scaling biosynthetic methods}

Expression of COMPcc was carried out in auto-inducing media, encompassing a
shift in protocols to lactose-rich media, in agreement with the methods refined
by Studier.\cite{Studier2005} Figure \ref{fig:AI_IPTG_gel} suggests that the
levels of incorporation are comparable on a cellular basis. More noteworthy is
the maximal growth level obtained via bioreactor expression, as shown in Figure
\ref{fig:bioreactor_OD_comparison}. Yields have been estimated to be in the
range of \SIrange{10}{20}{\mg\per\mL of culture}, prior to purification.
% --------------------------
\begin{figure}[h!]
    \centering
    \begin{subfigure}[b]{0.3\textwidth}
        \includegraphics[width=\textwidth]{f_2_12a}
        \caption{}
        \label{fig:AI_OD_curve}
    \end{subfigure}
    \begin{subfigure}[b]{0.3\textwidth}
        \includegraphics[width=\textwidth]{f_2_12c}
        \caption{}
        \label{fig:bioreactor_OD_comparison}
    \end{subfigure}
    \begin{subfigure}[b]{0.3\textwidth}
        \includegraphics[width=\textwidth]{f_2_12b}
        \caption{}
        \label{fig:AI_IPTG_gel} \end{subfigure}
    \caption[Summary of biosynthetic
    expresssion results]{Summary of biosynthetic expression results. (a) The
        auto-induction growth profile of BL21 (DE3) \emph{E.coli} transformed
        with pQE9-COMPcc. (b) Comparison of culture formats using auto-induction
        media, measuring turbidity via ${OD_{600}}$. Lanes 1-4 correspond to
        \SIrange{200}{250}{\mL} flask expressions grown for \SI{6}{\hour}, Lanes
        5-7 correspond to \SI{3}{\L} bioreactor expressions grown for 7, 9, and
        \SI{24}{\hour}, respectively. (c) SDS-PAGE analysis of cell culture
    lysate, comparing IPTG-inducible expression to that of
auto-induction.}
\label{fig:expression_results}
\end{figure}
% --------------------------


\subsection{Dosage}

\subsubsection{UPLC method development}
% --------------------------
\begin{figure}[h!] \centering \begin{subfigure}[b]{0.7\textwidth}
        \includegraphics[width=\textwidth]{f_2_13a} \caption{UPLC gradient for
        chromatographic separation} \label{fig:uplc_gradient} \end{subfigure}
    \begin{subfigure}[b]{0.7\textwidth}
        \includegraphics[width=\textwidth]{f_2_13b} \caption{Separation of BMS493, ATRA, and retinyl palmitate, collected at
            \SI{329}{\nm}, with noted retention times of \SI{2.176}{\minute},
            \SI{2.372}{\minute}, and \SI{7.151}{\minute}, respectively.}
        \label{fig:std_chromatogram} \end{subfigure}
    \begin{subfigure}[b]{0.7\textwidth}
        \includegraphics[width=\textwidth]{f_2_13c} \caption{Cross-sectional
            spectra
            at points of peak maxima, corresponding to ATRA, BMS493, and retinyl
        palmitate (top to bottom).}
        \label{fig:retinoid_spectra} \end{subfigure}
    \caption[Chromatographic separation of BMS493, ATRA, and retinyl
    palmitate]{Chromatographic separation of BMS493, ATRA, and retinyl palmitate
        using the UPLC gradient shown in (a). The resultant chromatogram (b)
        demonstrate successful separation of this control sample. From 3D
        spectra (data not shown), single retention time peak spectra were
    extracted for the identification of optimal wavelengths for
analytes.}\label{fig:uplc_report} \end{figure}
% --------------------------
UPLC method was optimized by first attempting separation of BMS493, retinoic
acid, and retinyl palmitate using a binary system of acetonitrile and water.
However, this resulted in poor resolution of retinoic acid. The addition of
methanol was then added, in agreement with previously reported
methods.\cite{DeLeenheer1982,Kane2008b,Wang2001a,Schaffer2010} However, BMS493
and retinoic were shown to co-elute easily with each other. To optimize the
resolution of BMS493 and endogenous retinoids, but also to maintain maximum
attainable peak heights (and signal/noise), a linear scouting gradient was
performed from 10:10:80 (ACN:MetOH:\ch{H2O}, \SI{0.1}{\percent} formic acid) to
45:45:10 (ACN:MetOH:\ch{H2O}, \SI{0.1}{\percent} formic acid) over
\SI{30}{\minute} (data not shown). From this optimization, maximal resolution
and minimum peak width were determined to be obtained using an isocratic mobile
phase consisting of 41.2:41.2:17.6 (ACN:MetOH:\ch{H2O}, \SI{0.1}{\percent}
formic acid). Elution of retinyl palmitate requires a relatively abrupt gradient
to \SI{100}{\percent} ACN, then held isocratically for approximately
\SI{3}{\minute} (Figure \ref{fig:uplc_gradient}). The cross-sectional spectra
presented in Figure \ref{fig:retinoid_spectra} allowed for single channels -
\SI{355}{\nm}, \SI{329}{\nm}, and \SI{324}{\nm} for ATRA, BMS493, and retinyl
palmitate, respectively - to be collected to obtain the best sensitivity of
detection as well as cross-channel calibration against retinyl palmitate.

\subsubsection{\latin{In vitro} availability}

To assess the availability of uptake of BMS493 \latin{in vitro}, liquid-liquid
extraction was applied to aqueous samples of BMS493. Test control extracts were
prepared at \SI{1}{\micro\moLar} in \SI{50}{\uL} of sample, consistent with the
operating parameters of the \latin{in vitro} cell culture experiments. Dried
organic phase extract was dissolved in ethanol and injected onto a reverse-phase
column as per the method set forth above. Levels were averaged across 5
replicates and compared against a benchmark of \SI{100}{\percent} extraction
efficiency, or a \SI{0.36405}{\ng} injection (Figure
\ref{fig:extract_controls}).
% --------------------------
\begin{figure}[h!]
    \centering
    \begin{subfigure}[b]{0.75\textwidth}
        \includegraphics[width=\textwidth]{f_2_14a}
        \caption{MilliQ, \SI{0.3366}{\ng}, \SI{92.46}{\percent} recovery}
        \label{fig:milliq_extract}
    \end{subfigure}
    \begin{subfigure}[b]{0.8\textwidth}
        \includegraphics[width=\textwidth]{f_2_14b}
        \caption{DMEM, \SI{0.335}{\ng}, \SI{92.02}{\percent} recovery}
        \label{fig:lonza_extract}
    \end{subfigure} \caption{UPLC quantitation of BMS493 (\SI{3.1}{\minute}) in
    extract controls prepared in different media. Each sample was spiked with
    \SI{0.1}{\volper} \SI{1}{\milli\moLar} BMS493 prepared in
DMSO}\label{fig:extract_controls} \end{figure}
% --------------------------
Extraction efficiency were approximately \SI{92}{\percent}.\footnote{A
    preliminary round of solid-phase extraction for two commercially available
    products - Sep-Pak \ch{C18} and Oasis columns (Waters Corporation) - (data
    not shown) was also carried out, resulting in \SI{18}{\percent} recovery
efficiencies.}
This extraction protocol was applied to a preliminary set of culture samples
from a BMS493 dosage study. This sample set varied in time, ATRA, and IL-1, with
constant BMS493 concentration of \SI{20}{\micro\moLar} (Table
\ref{tab:culture_extracts_table}). It is evident from the corresponding
chromatogram in Figure \ref{fig:culture_extracts_chrom} that the extraction was
bereft of any unknown impurities aside from the analytes of interest. IL-1 was
included in the study to additionally study the affect of cytokines on
osteoarthritis pathogenesis.
% --------------------------
%START_TABLE
%PRE-INDUCTION REAGENTS TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{ ccccc }
  \hline
  \multicolumn{5}{c}{Culture Sample Conditions} \\
  \hline
  Sample & Time (hr) & BMS493 & ATRA & IL-1 \\
  \hline

  1 & 0 & ${-}$ & ${-}$ & ${-}$ \\
  2 & 1 & ${+}$ & ${-}$ & ${-}$ \\
  3 & 1 & ${+}$ & ${+}$ & ${-}$ \\
  4 & 30 & ${+}$ & ${-}$ & ${+}$ \\
  5 & 30 & ${+}$ & ${+}$ & ${+}$ \\

  \hline
\end{tabular}
\caption[Culture sample conditions]{Culture sample conditions, taken from
overall dosage study, with single concentrations of BMS493, ATRA, and IL-1
(\SI{20}{\micro\moLar}, \SI{200}{\nano\moLar}, and \SI{10}{\ng},
respectively). Times correspond to the time elapsed after cell culture
initiation.}
\label{tab:culture_extracts_table}
\end{table}
%END_TABLE
% --------------------------
The peak about \SI{2.4}{\minute}, corresponding to BMS493, was isolated and
quantitated based on their peak areas against an external calibration of BMS493,
both sets of spectra internally calibrated against \SI{11}{\micro\moLar} retinyl
palmitate (Figure \ref{fig:culture_extracts_chrom_zoom}). The amount (\si{\ng})
of sample in the injection are plotted in Figure
\ref{fig:culture_extracts_chart} and labeled with the calculated \% uptake by
cells. Within \SI{1}{\hour} of incubation, the data suggests a marked decrease
in free BMS493 (\SI{60}{\percent} reduction), either due to cellular uptake or to degradative processes.
Furthermore, there appears to be synergistic uptake of BMS493 over time in the
presence of \SI{200}{\nano\moLar} ATRA, when comparing samples \textbf{4} and
\textbf{5}, which show a \SI{86}{\percent} and \SI{95}{\percent} reduction in
free BMS493, respectively. This synergistic uptake is also observed, but to a much lesser
extent, at the \SI{1}{\hour} time point, referring to samples \textbf{2} and
\textbf{3}, which differ by a minute \SI{3}{\percent}.
% --------------------------
\begin{figure}[h!]
    \centering
    \begin{subfigure}[b]{0.8\textwidth}
        \includegraphics[width=\textwidth]{f_2_15a}
        \caption{}
        \label{fig:culture_extracts_chrom}
    \end{subfigure}

    \begin{subfigure}[b]{0.45\textwidth}
        \includegraphics[width=\textwidth]{f_2_15b}
        \caption{}
        \label{fig:culture_extracts_chrom_zoom}
    \end{subfigure}
    \begin{subfigure}[b]{0.45\textwidth}
        \includegraphics[width=\textwidth]{f_2_15c}
        \caption{}
        \label{fig:culture_extracts_chart}
    \end{subfigure}
    \caption[UPLC quantitation of BMS493 in samples from an \latin{in vitro} dosage
    study]{UPLC quantitation of BMS493 in samples from an \latin{in vitro} dosage
    study. Sample \textbf{1}: \SI{0}{\hour}, ${-}$BMS493, ${-}$ATRA, ${-}$IL-1;
    \textbf{2}: \SI{1}{\hour}, ${+}$BMS493, ${-}$ATRA, ${-}$IL-1; 
    \textbf{3}: \SI{1}{\hour}, ${+}$BMS493, ${+}$ATRA, ${-}$IL-1; 
    \textbf{4}: \SI{30}{\hour}, ${+}$BMS493, ${-}$ATRA, ${+}$IL-1; 
    \textbf{5}: \SI{30}{\hour}, ${+}$BMS493, ${+}$ATRA, ${+}$IL-1; 
}\label{fig:uplc_report_culture_extracts}
\end{figure}
% --------------------------

%\subsection{Screening of agonists/antagonists}

%\subsection{Small molecule stability}
%Introduction to light sensitivity of molecules

%\subsubsection{Light-induced oxidation}
%DATA: initial chromatography studies 

%\subsubsection{\latin{In vitro availability}}
%DATA: extraction method development
%Initial attempt from GM20130716
%Second attempt via SPE GM20130913
%Final attempt, including curves from last MM report.
%DATA: method application to BMS493 dosage study


\subsection{Small molecule binding}
The ability to bind small molecules was first assessed using ATRA, a close
analogue of \iupac{all-\trans-retinol}, which has already been shown to bind to
COMPcc with \si{\micro\moLar}
affinities.\cite{Guo1998,Haghpanah2010,Gunasekar2009} The method of zonal
elution chromatography was assessed as to whether it would be appropriate and
sufficient for further application to BMS493. Briefly, COMPcc and BMS493 were
mixed and incubated in the sample loop chamber of an \"{A}KTA purifier system,
awaiting successive periodic injections as shown in Figure
\ref{fig:zonal_elution_report}. The chromatograms suggest that the protein is
eluting within the first \SI{50}{\percent} of the column's void volume,
consistent with the typical migration profile of macromolecules through G-25
media. Interestingly, however, there is a shift in both intensity and retention
time of the elution curve, as charted in Figure \ref{fig:ze_intensity} and
\ref{fig:ze_retention_times}. The increase in signal intensity may be due to the
superposition of ATRA signal with the protein elution profile. Interestingly,
Figure \ref{fig:ze_retention_times} suggests an enlargement of the
supramolecular complex that is forming while the sample resides in the super
loop over time, as indicated by the shorter retention times of the bulk of the
sample. It should be noted, however, that the increase in intensity may not be
due to a superposition of ATRA signal, which in itself is comparatively weak
in the control injections, but may rather be due to macroscopic assembly,
resulting in light scattering within the flow cell detector. This would,
however, further confirm the conclusion that the protein is self-assembling into
larger structure over time.
% --------------------------
\begin{figure}[h!]
    \centering
    \begin{subfigure}[b]{0.75\textwidth}
        \includegraphics[width=\textwidth]{f_2_16a}
        \caption{}
        \label{fig:zonal_elution_chrom}
    \end{subfigure}

    \begin{subfigure}[b]{0.4\textwidth}
        \includegraphics[width=\textwidth]{f_2_16b}
        \caption{}
        \label{fig:ze_intensity}
    \end{subfigure}
    \begin{subfigure}[b]{0.4\textwidth}
        \includegraphics[width=\textwidth]{f_2_16c}
        \caption{}
        \label{fig:ze_retention_times}
    \end{subfigure} \caption[Zonal elution profile for ATRA mixed with
    COMPcc]{(a) Zonal elution profile for COMPcc mixed with ATRA as a 1:1
        mixture of \SI{25}{\micro\moLar} prepared in \SI{100}{\milli\moLar}
        Gomori phosphate buffer, pH 8.0, \SI{10}{\volper} ethanol. Injections
        were \SI{1}{\mL} loaded onto a \SI{5}{\mL} HiTrap G-25 Desalting column.
        The dashed line curve corresponds to an initial injection at
        time=\SI{0}{\hour}. Darkening curves indicate an increase in incubation
        time from \SI{1}{\hour} to \SI{18}{\hour}. (b) Signal intensity increase
        at the peak absorption of each successive run. (c) Peak retention
    time/volume shift of each successive run.}\label{fig:zonal_elution_report}
\end{figure}
% --------------------------
%DATA: zonal elution chromatography

%First attempts with ATRA as a control molecule. Unfortunately no dice, despite
%the promising results, it is probably due to light scattering effects.
%DATA: Hummel Dreyer chromatography

%\subsection{Small molecule release}


\section{Completion Pending}

As these studies are currently in the process of completion, any discussion and
conclusions based on this work have yet to be drafted. It can be stated,
however, that based on the zonal elution profiles collected thus far, the better
method to entertain is that of Hummel Dreyer chromatography. As a more robust
method to sample dilution through the column, it will yield more relevant
information regarding the binding characteristics of the protein to BMS493. More
interesting is the potential dual outcome of committing such an experiment on an
oligomeric acceptor species (COMPcc), yielding information not just of the
binding of BMS493 onto COMPcc, but also the binding of BMS493 to particular
oligomeric species as they resolve themselves through the column. This would
yield potential insight as to whether the full coiled-coil is necessary for
binding, and perhaps separate the engineering goals of optimizing full
pentameric coiled-coil assembly and binding affinities to small molecules.

%\section{Conclusion}

\printbibliography[heading=subbibliography]

\end{refsection}
