\chapter{Modulation of assembly properties of protein-based biomaterials via
global fluorinated amino acid incorporation}

\begin{refsection}

\section{Introduction}

\subsection{Protein-based materials}

% Introduce proteins as materials - a sort of history survey of bio-based
% functional material and their corresponding various application fields.

Genetically engineered proteins have made a significant impact in biomaterials
research, enabling the fabrication of polymers with monodisperse molecular
weights, a diverse set of side chain functionality, and well-defined secondary
structure elements.\cite{
Yoshikawa1994,Wang2001,Rabotyagova2009,Martin2009,Dinerman2002,Megeed2002,Haider2004,Wright2002,Banta2010,Link2003,Straley2009,Rabotyagova2011}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{f_1_03}
    \caption[The predominant methodologies of protein
        engineering.]{The predominant methodologies of protein
        engineering.\cite{VanHest2001}}\label{fig:protein_engineering_methods} \end{figure}
% --------------------------
Compared against a backdrop of predominant synthetic polymers, protein polymers,
particularly those generated via high-fidelity recombinant DNA technologies,
exhibit defined molecular weight, composition, sequence, and
stereochemistry.\cite{VanHest2001} While proteins are not alone amongst
engineered materials of biological origin, they exist as the intersection of
biological function and structure. The latter feature spans the gamut of length
scales, ranging from the nanoscopic to the macroscopic, with notable examples
including actin filaments in the intracellular milieu to the silk fibers visible
to the naked eye.\cite{Rabotyagova2009,Megeed2002,Nagarsekar2002} Their frequent
roles as both sensors and chemical catalysts are testaments to their functional
impact on nature.

% Begin to focus on example of proteins that have secondary structural motifs
% that abstract themselves to higher order structures, such as fibers.
% Generalize how these can span length scales from nanoscopic order to
% macroscopic order.  Why do these higher order structures typically form. 

Numerous examples exist of proteins extrapolating the assembly process beyond
the quaternary scale of folding. This arena of self-assembly is of particular
focus herein, requiring inquiry into the evidence of such phenomenon occurring,
and more importantly, how these self-assembly events occur mechanistically.

In the case of collagen, secondary structure is established as alpha-helices but
further progresses to the self-association of monomers to form triple helices.
More interestingly, this tiered assembly, wherein the larger macromolecular
assemblies become dependent on the secondary structure of monomers imbues the
opportunity for sequence-dependent specificity of assemblies. Indeed this
inherent feature of these self-assembling systems has been leveraged by several
engineering design groups.\cite{}

Similarly, resilin, the proteinaceous material responsible for the elasticity of
various insect wings and appendages, relies on a critical proline residue to
accommodate regular turns in the protein backbone that promote a molecular
spring able to survive innumerable duty cycles.\cite{}

In the case of elastin-based peptides, secondary structure is established as a
preliminary phase of assembly. The prominent theory in the field dictates that
the higher order self-assembly of these class of peptides is governed by the
hydrophobic collapse of the macromolecule. In certain cases, this results in a
macroscopic network of peptides to yield either a gel or colloid of
coacervates.\cite{}

Silk is another prominent natural protein on which novel biomaterial design is
based. It exists as an example of a heterogeneous mixture of structured and
unstructured secondary structures.  $\beta$-sheets dominate the ordered features
of silk variants.

\subsection{Block protein polymers}

\subsubsection{Fusion Proteins}

\subsubsection{Elastin-like peptides}

% Introduce the concept of supramolecular assembly.
The lazy brown fox.

\subsubsection{Cartilage oligomeric matrix protein}

The lazy brown fox.

\subsubsection{Block ELP-COMP Protein Polymers}

The lazy brown fox.
% --------------------------
\begin{figure}[h!]
    \centering
    \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{Schema indicating three block copolymer proteins that were
        previously generated, via recombinant DNA techniques, and incorporated
        with \iupac{\para-fluorophenylalanine} via residue-specific incorporation.
    }
    \label{fig:pff_proteins}
\end{figure}
% --------------------------
\subsection{Incorporation of non-natural amino acids}

While the physicochemical properties of protein-based materials have been
modulated with extraordinary capacity and range\cite{} - in large part due to
the non-covalent interplay of amino acid residues - they are still limited by
the constrained 22 amino acids that define functional group diversity.\cite{}
This limitation has been addressed, and in many ways deprecated, by various
efforts in genetic engineering.\cite{} 

There exists two contemporary methods to biosynthetically incorporate
non-natural amino acids into proteins: site-specific incorporation and
residue-specific incorporation.

\subsubsection{Site-specific incorporation methods} 

% Document the example of site-specific incorporation of azido-containing amino
% acids into polypeptides.

Early work in the mechanistic studies, and resultant ``hacking'' of the
components of the natural protein synthesis machinery, yielded the eventual
maturity of incorporation techniques.  Site-specific incorporation has been a
viable strategy for the incorporation of non-natural amino acids into
polypeptides since the late 1990's, yet its development stemmed from
advancements occurring as early as the late 1970's, with the work of Sidney
Hecht on the chemical aminoacylation of tRNA and subsequent work on the
misacylation of tRNA{$^\text{Phe}$}.\cite{Heckler1984,Hecht1978} This later
fostered incremental efforts of Brunner in the charging of tRNA{$\text{Phe}$}
with \iupac{\L-4'-[3-(trifluoromethyl)-3\H-diazirin-3-yl]}phenylalanine, which
still required the chemically misaminoacylation of tRNA in the presence of T4
RNA ligase and ATP.\cite{Baldini1988}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{f_1_05}
    \caption[Strategy for the site-specific incorporation of unnatural amino
    acids.]{Strategy for the site-specific incorporation of unnatural amino
    acids.\cite{Noren1989}}\label{fig:schultz_schema} \end{figure}
% --------------------------
With the seminal efforts of the Schultz \latin{et al.}\cite{Noren1989} shortly
following these earlier bodies of work, it is now possible to redefine the
genetic code. 
% --------------------------
Years later, in 2001, the Schultz group reported on the development of a unique
transfer RNA (tRNA)/aminoacyl-tRNA synthetase pair that were genetically
engineered to exist as orthongal complments to their endogenous analogues, as
well as associate the encoding for the amber nonsense codon with a non-natural
amino acid - \iupac{\ortho-methyl-\L-tyrosine}. The group was able to thus
incorporate a non-natural, synthetic amino acid with notably high fidelity
(99\%).\cite{Wang2001} This method relies on the translation suppression of
endogenous amber nonsense codons, which is addressed by the orthogonal nature of
the system's design. Similarly, Kwon and coworkers have been able to reassign
one of the degenerate codons (UUU) that correspond to phenylalanine to encode
for \iupac{2-naphthylalanine}.\cite{Kwon2003}

This technique has been employed in various contexts, ranging from the study of
the structure-function relationship of enzymes and receptors, to the rationale
design of proteins with higher activities or structural stabilities.
%How site-specific incorporation has been used to probe structure-function
%relationships

%How site-specific incorporation has been used to rationally design proteins

%Other groups have pushed the envelope still further with this technique,
%reimagining the genetic code itself.

\subsubsection{Global incorporation methods} 

% Include an argument in support of residue-specific incorporation via
% endogenous tRNA/aminoacyl-tRNA synthetase

Global incorporation of non-natural amino acids is further made possible by the
particular technique of residue-specific substitution. This technique is
typically characterized by a stochastic substitution of all amino acids of a
particular identity within a given sequence. It is made possible by two
prominent factors: 1) the promiscuity and relaxed specificity of aminoacyl-tRNA
synthetase endogenous to certain expression hosts, and 2) the existence of
auxotrophic expression hosts.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{f_1_02}
    \caption[Terminal pathway of phenylalanine
        biosynthesis.]{Terminal pathway of phenylalanine
        biosynthesis.\cite{Yoshikawa1994}}\label{fig:chorismate_pathway} \end{figure}
% --------------------------
\subsection{Fluorinated amino acids}

% What's the earliest precedence found for fluorinated amino acids and the
% motivation behind their synthesis?


\subsection{Fluorous effect}

The lazy brown fox.

\subsection{Scope of work}

The primary goals of this work were to evaluate the incorporation of
the fluorinated amino acid, \iupac{\para-fluorophenylalanine}, into protein block
copolymers that were previously developed, and to evaluate the effects of this
incorporation on the physicochemical properties of the polymers.
% --------------------------
It is hypothesized that the substitution of fluorine onto the guest residue of
the elastin-like peptide block sequence will significantly affect the behavior
of the bulk polymer, supported by the current understanding of elatin-like
peptides, elastin-like peptide fusions, and precedent observations of
fluorinated biomacromolecules.

% Guest residue effects by Urry.
% Chilkoti's block influence work.
% Fluorous effect.

To probe the effects on the physicochemical behavior of these block copolymers,
three primary experiments were conducted, spanning various scales of structure -
circular dichroism spectroscopy, UV/Vis spectroscopy, and microrheology.
Circular dichroism was implemented in order to interrogate the 

\subsubsection{Microrheology}

While several methods exist to evaluate the mechanical properties of fluids,
colloidal suspensions, and polymeric gels, microrheological techniques are
necessary to evaluate samples that are limited in volume and material
homogeneity. Classical oscillatory plate and capillary cell rheometers could not
accommodate the anticipated nature of block protein polymers, with expectedly
low viscoelastic moduli and low response frequencies. Micorheological methods,
however, could be employed to observe the mechanical properties of dilute
supramolecular assemblies. These methods include the application
fluctuation-dissipation theorem-based (FDT)
microrheology,\cite{Gittes1997,Mason1997,Mason1997a} two-point microrheology
(TPM),\cite{Crocker2000,Levine2000,Qiu2004,Levine2001} and atomic force
spectroscopy.\cite{Salman2002,Caspi2002} Herein, we employ a form of FDT-based
microrheology, which may be accomplished via a myriad of instrumental
techniques, including laser-deflection tracking,\cite{Gittes1997,Mason1997}
optical tweezers,\cite{Starrs2003,Addas2004} dynamic light
scattering,\cite{Dasgupta2002,Popescu2002} and image-based particle
tracking\cite{Valentine2004,Crocker2000,Chen2003}- all methods toward an effort
to quantify the Brownian motion of particles suspended in the rheological fluid
of interest.

The particular application of particle tracking implemented hails from 

\section{Methods}

\subsection{Recombinant gene construction}

Plasmid vectors containing the genes for the block protein polymer fusions were
created in a modular fashion, described in a previously accomplished body of
work.\cite{Haghpanah2009} Briefly, PCR assembly of the COMP gene and
flanking linker regions yielded a pQE9 vector (Qiagen) contianing the final
single block construct. This was then digested with SalI and SacI restriction
enzymes, and gel purified for insertion into a pQE30 plasmid vector, yielding
pQE30/C. 

The 5-repeat elastin gene was PCR amplified from pUC19RGDELF, donated by David
Tirrell, and cloned into pQE30/C between various restriction sites, to yield
pQE30/EC (BamHI and SacI), pQE30/CE (SalI and HindIII), and pQE30/ECE (all 4
sites).

\subsection{Biosynthesis}

The aforementioned plasmid vectors were transformed into the phenylalanine
auxotrophic \emph{E. coli} strain AF-IQ, previously created and donated by David
Tirrell \emph{et al}.\cite{Yoshikawa1994,} Cells growth methods were based on
previously established conditions for the incorporation of unnatural amino
acids, particularly pFF, into target
proteins.\cite{Voloshchuk2009,Yoshikawa1994}

Plasmids were transformed into AF-IQ cells via electroporation using the
following parameters on an electroporator (BioRad, Inc.) The cells were then
plated onto a tryptic soy agar plate, 40
\si{\g\per\L},
supplemented with ampicillin (200 
\si{\ug\per\mL})
and chloramphenicol (35
\si{\ug\per\mL}), and
allowed to grow overnight at 37 \si{\celsius}. Single colonies were then picked
and used to inoculate starter cultures, grown in M9 minimal media, supplemented
with the following components:

%START_TABLE
%PRE-INDUCTION REAGENTS TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{ ll }
  \hline
  \multicolumn{2}{c}{Component (Final Concentrations)} \\
  \hline

  ampicillin & \SI{200}{\ug\per\mL} \\
  chloramphenicol & \SI{200}{\ug\per\mL} \\
  thiamine hydrochloride & \SI{200}{\ug\per\mL} \\
  magnesium sulfate & \SI{200}{\ug\per\mL} \\
  calcium chloride & \SI{200}{\ug\per\mL} \\
  glucose & \SI{200}{\ug\per\mL} \\
  20 amino acids & \SI{1}{\ug\per\mL} \\

  \hline
\end{tabular}
\caption{Pre-induction M9 Supplement Specifications}
\label{tab:preinduction_recipe}
\end{table}
%END_TABLE

Cells were grown in M9 supplemented media in two stages - a pre-induction stage
and a post-induction stage - which are separated by a media shift step.  The
starter culture was used to inoculate a larger \SI{200}{\mL} culture, as
\SI{4}{\volper}, grown in pre-induction M9 media as per Table
\ref{tab:preinduction_recipe}.  The culture was then allowed to grow in a
\SI{1}{\L} flask for \SI{6}{\hour} at \SI{37}{\celsius}, shaking at
\SI{350}{\rpm}. The culture was subsequently harvested and pelleted via
centrifugation. The pellet was subjected to resuspension in a volumetric
equivalent of \SI{0.9}{\wtper} NaCl solution, followed by centrifugation.  This
``wash'' step was performed 2 times in an effort to deplete the media of
residual amounts of extracellular phenylalanine. The washed pellet was
resuspended in fresh M9 supplemented media.  The distinct difference between the
pre- and post-induction supplemental reagents pertain to the amino acids
constitution; the post-induction media does not contain phenylalanine, but
rather contains \SI{91.59}{\mg\per\mL} \iupac{\para-fluorophenylalanine},
prepared from a stock.  Cells in the new media were allowed to grow at the same
incubation conditions for \SI{30}{\minute}, at which time the media was
supplemented with \iupac{isopropyl \b-\D-1-thiogalactopyranoside} (IPTG,
\SI{100}{\ug\per\mL}).  Cells were harvested and stored at -20 \si{\celsius} for
subsequent lysis and purification.

\subsection{Protein purification}

All solutions used in the extraction and purification of recombinant proteins
were filtered through \SI{0.22}{\um} sterile filters. In addition, all solutions
were adjusted to the appropriate \pH with either \SI{37}{\percent} \ch{HCl} or
\SI{1}{\moLar} \ch{NaOH}. Cell pellets were thawed at \SI{4}{\celsius} and
immediately subected to an osmotic shock protocol adapted from methods set forth
by Berglund \emph{et al}.  for the preparation of cell lysate for immobilized
metal affinity chromatography (IMAC)
purification.\cite{Neu1965,Bae2000,Magnusdottir2009} Briefly, cells were gently
resuspended in a cold hypertonic sucrose solution, consisting of
\SI{50}{\milli\moLar} HEPES, \SI{20}{\wtper} sucrose, \SI{1}{\milli\moLar}
ethylenediaminetetraacetic acid (EDTA), \pH 7.9. For every \SI{10}{\mL} of
original culture media, the resultant pellet was resuspended in \SI{1}{\mL} of
sucrose solution. The cell suspension was then pelleted via centrifugation at
\SI{7000}{\gforce} for \SI{30}{\minute} at \SI{4}{\celsius}. The resultant
pellet was separated from its supernatant and gently resuspended in a cold
hypotonic solution consisting of \SI{5}{\milli\moLar} \ch{MgSO4}, with a
volumetric equivalent to that of the previous step, and incubated on ice for
\SI{10}{\minute}. The suspension was then centrifuged at \SI{4500}{\gforce} for
\SI{20}{\minute} at \SI{4}{\celsius}. The resultant pellet was separated from
its supernatant and stored at \SI{-20}{\celsius} for downstream lysis. 

Osmotically shocked expression pellets were resuspended in lysis buffer,
consisting of \SI{50}{\milli\moLar} \ch{Na2HPO4}, \SI{6}{\moLar} \ch{CO(NH2)2},
\pH 8.0. The pellet was resuspended in a 4:70 volumetric ratio (buffer:culture
volume) on ice. The cell lysate was then passed through a French press
(40K-cell; Thermo Scientific), processed at \SI{165.5}{\MPa}, and collected on
ice. The cold lysate was then centrifuged at \SI{40000}{\gforce} for
\SI{45}{\minute} at \SI{4}{\celsius}. The clarified lysate was then applied to a
\SI{5}{\mL} HiTrap IMAC FastFlow column (GE Life Sciences), charged with
\ch{CoCl2}, using an \"{A}KTA purifier system (GE Life Sciences). The lysate was
loaded with \SI{0}{\moLar} imidazole and washed with 10 column volumes
(CV) of the same buffer constitution. The protein was then eluted across 1 CV of
\SI{48}{\milli\moLar} imidazole and 3 CV of \SI{1}{\moLar} imidazole.  Purified
elutions were assayed for purity via sodium dodecylsulfate polyacrylamide gel
electrophoresis (SDS-PAGE) analysis and, contingent upon confirmation of
\SI{90}{\percent} purity, was dialyzed extensively against water with a 3500
MWCO regenerated cellulose membrane (SnakeSkin Dialysis Tubing, Pierce) and
subsequently freeze-dried prior to use in experiments.

\subsection{Compositional characterization}
\subsubsection{MALDI-ToF mass spectrometry}

Incorporation levels of pFF were assessed via matrix-assisted laser desorption
ionization, time-of-flight mass spectrometry analysis on an Omniflex
spectrometer (Bruker Daltonics). Chymotryptic digests of purified proteins were
prepared by resuspending \SIrange[range-phrase=--]{50}{100}{\ug} of protein in
\SI{100}{\milli\moLar} Tris-HCl and \SI{10}{\milli\moLar} \ch{CaCl2} (\pH 8.0)
to a final concentration of \SI{1.25}{\ug\per\uL}. Sequencing grade chymotrypsin
(Promega) was added to the mixture to yield a final concentration of
\SI{2}{\wtper}. The digestion reaction was allowed to occur at \SI{25}{\celsius}
for \SI{20}{\hour} and subsequently stopped by adjusting the mixture to \pH 4 by
titrating with \SI{1}{\percent} trifluoroacetic acid (TFA). The protein digest
mixtures were desalted and concentrated by using ZipTip(\ch{C18}) silica resin
(Millipore).  Desalted sample mixtures were mixed with saturated solutions of
\iupac{\a-cyano-4-hydroxycinnamic acid} prepared in \SI{1}{\percent}
TFA/acetonitrile (2:1) in a 1:1 ratio. Samples were then spotted onto a 7x7
OmniFlex MALDI target (Bruker Daltonics) and allowed to dry under vacuum.
Relative estimates of peptide digest fragment quantities were conducted by
comparing relative peak intensities at different \emph{m/z} values. MS spectra
were calibrated against a standard mixture of peptides, provided by New England
Biolabs, consisting of \ldots

\subsubsection{Amino acid analysis}

Amino acid analysis was performed on purified protein samples that have
undergone hydrolysis. Hydrolysate samples were then analyzed on a Hitachi L-8900
pH amino acid analyzer at the W. M. Keck Foundation Biotechnology Resource
Laboratory at Yale University. Incorporation levels of pFF were calculated based
on the measurements of normalized Phe levels in the samples.

\subsection{Physicochemical characterization}

\subsubsection{UV/Vis spectroscopy}

UV/vis spectroscopy was performed on a Cary 100 UV/vis spectrometer (Agilent) to
determine the inverse temperature transition point ({$T_t$}) of 
\SI{4}{\micro\moLar} protein prepared in \SI{10}{\milli\moLar} Gomori buffer,
\pH 8.0. The absorbance at \SI{350}{\nm} was measured as each protein sample was
heated from \SIrange{10}{65}{\celsius}, at a heating rate of
\SI{0.5}{\celsius\per\minute}.\cite{Chilkoti2002a}

\subsubsection{Circular dichroism spectroscopy}

Circular dichroism (CD) spectroscopy was carried out on a J-815 CD polarimeter
(Jasco) equipped with a PTC-423S single position Peltier temperature control
system and cuvette holder, counter-cooled with an Isotemp 3016S water bath
(Fisher Scientific) set to \SI{22}{\celsius}. Protein samples were prepared to
\SI{4}{\micro\moLar} in \SI{10}{\milli\moLar} Gomori buffer, by resuspending
lyophilized protein and loading into a Helma 218 quartz cuvette (\SI{500}{\uL}
capacity, \SI{1}{mm} path length). Immediately prior to loading into the
spectrometer, samples were degassed for \SI{10}{\minute} by sparging samples
with nitrogen. Operation and analysis parameters were adapted from existing
procedures.\cite{Haghpanah2009} Wavelength scans were performed while holding
temperature constant as the samples were heated from \SIrange{10}{65}{\celsius},
at a heating rate of \SI{1}{\celsius\per\minute}.  Wavelength scans were
performed from \SIrange{250}{190}{\nm} with a \SI{1}{\nm} step size. Wavelength
scans were collected as the average of 3 consecutive scans, with a preliminary
equilibration time of \SI{5}{\minute} prior to the first scan. All wavelength
scans were subtracted from the background of Gomori buffer.

Mean residue ellipticity, $[\theta]_{\mathrm{mrw}}$ is calculated from
expression \ref{eq:mre} below:

\begin{equation}
    \label{eq:mre}
    [\theta]_{\text{mrw}}=
    \frac{[\theta]_{\text{obs}}\cdot\text{MRW}}{10\cdot{C_\text{mg/mL}}\cdot{L}}
    \text{ (units : \si{\degtext\cm\squared\per\deci\mol})}
\end{equation}

where $[\theta]_{\text{obs}}$ is the observed ellipticity signal, MRW is the
mean residue weight (molecular weight divided by the number of residues), $L$ is
the path length of the cell (in cm), and $C_\text{mg/mL}$ is the protein's
concentration.\cite{Martin2008} MRE spectral data was smoothed using a
Savitsky-Golay low-pass filter, applied with SpectraManager software package (Jasco),
using a smoothing width of 25. Estimation of secondary structure was performed
computationally using the CDSSTR method, as distributed by DICHROWEB (Birkbeck
College of the University of London). The method was applied with the SDP49
protein reference set.
\subsubsection{Microrheology}

To interrogate the mechanical properties of the proteins, passive microrheology
was performed using an inverted Leica DM-138 IRB microscope with a 20$\times$
objective lens at \SI{22}{\celsius} and \SI{42}{\celsius}, maintained by a
Linkam temperature-controlled microscope stage. Samples were prepared from
lyophilized proteins by dissolving dry powder in deionized water at
\SI{4}{\celsius} to two concentrations, \SI{1.25}{\mg\per\mL} and
\SI{2.5}{\mg\per\mL}. \SI{2}{\uL} of a \SI{2}{\wtper} stock of
fluorescently-labeled amidated polystyrene beads (\SI{1.0}{\um} was added to
\SI{10}{\uL} samples of protein. Brownian motion of the beads was observed by a
Peltier-cooled video camera (QiCam) and digitally recorded.  Mean square
displacement (MSD) measurements of the beads, dispersed in the protein sol gels,
were determined using IDL software (Research Systems, Inc.) and an application
of microrheological processing subroutines set forth by Mason and
Weitz.\cite{Mason1997} The subroutines as well as the parameters which they are
fed, are documented in Appendix B. 

\section{Results}

\subsection{Biosynthesis of Fluorinated Block Polymers}

Residue-specific incorporation of pFF was accomplished using previously
established techniques by the Montclare Group, and previously by the Tirrell
Group.\cite{Yoshikawa1994,Sharma2000,Voloshchuk2009} Three primary methods of
incorporation analysis were conventionally practiced at the time of this work:
SDS-PAGE, amino acid analysis (AAA), and mass spectrometry (MS). 
%Cite comparative works that use this overall approach of 3 methods.
This regimen of analysis for incorporation of non-natural amino acids has been
recently implemented by Montclare, Baker, and
Voloshchuk\cite{Voloshchuk2009,Panchenko2006,Baker2011}. In
addition, several groups have implemented AAA as part of their evaluation of novel
peptides. MALDI-MS, due to its limited quantitative capacity, has been
implemented to a lesser extent, but still has been employed in some
studies.\cite{Taki2001}

SDS-PAGE analysis of \emph{E.coli} expression lysate samples were carried out to
confirm proper control of induction via the \emph{lac} operon of the pQE-30
expression vector, as well as the suppression of significant amounts of protein
expression in the absence of \iupac{phenylalanine}.  Figure
\ref{fig:expression_gel_pFF} indicates the sufficient control of expression so
as to achieve effectively sole incorporation of \emph{p}FF. The absence of
expression in the absence of IPTG and the absence of either Phe or \emph{p}FF
suggests a tight control of the inducible expression system. This is consistent
with the complementary design of the expression host/vector pairing - AF-IQ and
pQE-30 - of this implementation.

More specifically, in the presence of Phe or \emph{p}FF, a distinct band was
observed at \SI{25}{\kilo\dalton} for the diblocks and \SI{37}{\kilo\dalton} for
the triblock, confirming the production of protein polymers. Similar results
were obtained from previous studies, of equivalent differentials in band
intensities so as to conclude suppression of expression.
%Cite comparative SDS-PAGE results
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.5\textwidth]{f_1_04}
    \caption[Confirmation of \emph{p}FF-EC, \emph{p}FF-CE, and \emph{p}FF-ECE
    proteins, SDS-PAGE of whole cell lysate samples] {Confirmation of
        \emph{p}FF-EC, \emph{p}FF-CE, and \emph{p}FF-ECE proteins, SDS-PAGE of
        whole cell lysate samples (normalized to OD{$_{600}$} = 1.0) in the
        absence and presence of phenylalanine and \emph{p}FF. Lanes 1, 5, and 9:
        preinduction lysate; 2, 6, and 10: -Phe/-\emph{p}FF postinduction
        lysate; 3, 7, and 11: +Phe/-\emph{p}FF postinduction lysate; 4, 8, and
        12: -Phe/+\emph{p}FF postinduction lysate. Arrows indicate protein
        overexpression band. The absence of the overexpression bands in lanes 1,
        2, 5, 6, 9, and 10 indicates that the expression methodology does not
        suffer from ``leaky'' expression of protein, which would result in the
        significant contamination of Phe in the purified protein. Further, the
        comparable intensity of paired bands in lanes 3, 4, 7, 8, 11, and 12
        suggests that the expression method is able to readily support the
        presence and incorporation of \emph{p}FF in the system.}
        \label{fig:expression_gel_pFF} \end{figure}
% --------------------------
AAA and MS serve as orthogonal means to assess incorporation, with each method
providing its own merits and succumbing to its respective limitations. While AAA
provides high-fidelity quantitation of the amino acid composition of a protein,
it comes at the expense of complete hydrolysis of the sample and resultant loss
of primary sequence information. MS of enzymatic-digest fragments of the
protein, on the other hand, may only be considered quasi-quantitative under the
assumption of equivalent propensities of fragments to ionize, despite the
difference in primary structure and molecular weight; however, under this
assumption, it may be used to elucidate the relative incorporation levels of a
non-natural amino acid across different segments of the protein, allowing the
researcher to conclude an homogeneous dispensation of non-natural amino acid
throughout the primary sequence of the protein.

%START_TABLE
%Incorporation Levels TABLE
\begin{table}[h!]
    \centering
\begin{tabular}{lcc}
  \hline
  \multicolumn{3}{c}{Incorporation Levels} \\
  \hline
  Protein Construct & AAA & MALDI-ToF MS \\
  \hline
  pFF-EC & \SI{81.00(12)}{\percent} & \SI{89(5)}{\percent} \\
  pFF-CE & \SI{77.75(29)}{\percent} & \SI{87(7)}{\percent} \\
  pFF-ECE & \SI{88.85(100)}{\percent} & \SI{94(3)}{\percent} \\
  \hline
\end{tabular}
\caption{Incorporation levels as measured by Amino Acid Analysis (AAA) and
MALDI-ToF MS Analysis, accompanied by associated standard deviations for
respective measurements.}
\label{tab:incorporation_numbers}
\end{table}
%END_TABLE

MALDI-ToF analysis of chymotryptic digests of each of the proteins was carried
out. Figures \ref{fig:MALDI_spectrum_EC}, \ref{fig:MALDI_spectrum_CE}, and
\ref{fig:MALDI_spectrum_ECE} delineate the main two fragments that result from
chymotrypsin digestion, which cleaves the protein at Phe residues.
Fortuitously, the enzyme appears to easily cleave sites that display \emph{p}FF
as opposed to Phe. Each spectrum indicates a shift in mass-to-charge ratio that
results from the shift imparted by an additional \SI{18}{\dalton}.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{} \label{fig:MALDI_spectrum_EC} \end{figure}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{} \label{fig:MALDI_spectrum_CE} \end{figure}
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{} \label{fig:MALDI_spectrum_ECE} \end{figure}
% --------------------------
%Cite comparative incorporation results
The discrepancy between AAA and MALDI-MS estimations for incorporation,
\SIrange[range-phrase=--]{6}{11}{\percent}, are slightly higher than that
observed previously for the incorporation of \emph{p}FF into tGCN5, but on the
same order of magnitude.\cite{Voloshchuk2009}
% --------------------------
%Implications of incomplete incorporation

The lazy brown fox.

\subsection{Secondary Structural Anaylsis}

The lazy brown fox.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{} \label{fig:CD_temp_wl} \end{figure}
% --------------------------
The lazy brown fox.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{} \label{fig:CD_computation} \end{figure}
% --------------------------
The lazy brown fox.

\subsection{Thermoresponsive Supramolecular Assembly}

The lazy brown fox.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{} \label{fig:lcst} \end{figure} k The lazy brown fox.
% --------------------------

\subsection{Mechanical Behavior}

The lazy brown fox.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.8\textwidth]{f_1_01}
    \caption{} \label{fig:rheology} \end{figure}
% --------------------------
The lazy brown fox.

\section{Discussion}

The lazy brown fox.

\section{Conclusion}

The lazy brown fox.

\printbibliography[heading=subbibliography]

\end{refsection}
